Description: Remove binary ROM dumpers and display error message.
 As software in Debian 'main' must not require or recommend a package
 outside of main for compilation, the pre-compiled ROM dumpers are
 removed by this patch and an error message is displayed when trying
 to dump the ROM. 
 Cf. <URL:https://lists.debian.org/debian-mentors/2013/08/msg00196.html>
 for more details on the issue.  The original patch has been prepared by
 Benjamin Moody <benjamin.moody@gmail.com>.  It has been tweaked heavy. 
Author: Benjamin Moody <benjamin.moody@gmail.com>
Forwarded: no
Reviewed-by: Andreas B. Mundt <andi@debian.org>
Last-Update: 2013-08-21

--- a/src/error.c
+++ b/src/error.c
@@ -243,6 +243,14 @@
 		*message = g_strdup(error_msg);
 		break;
 
+	case ERR_ROMDUMP_UNINSTALLED:
+		strcpy(error_msg, _("ROM dumping program not found."));
+		strcat(error_msg, "\n");
+		strcat(error_msg,
+		_("Cause: ROM dumps are not possible with this program because the tools needed to build the ROM dumpers are not included in Debian."));
+		*message = g_strdup(error_msg);
+		break;
+
 	case ERR_CALC_ERROR1:	// must be synchronized with cmd89.c (static uint8_t dbus_errors[])
 		strcpy(error_msg, _("Msg: hand-held returned an error."));
 		strcat(error_msg, "\n");
--- a/src/calc_84p.c
+++ b/src/calc_84p.c
@@ -654,7 +654,7 @@
 
 static int		dump_rom_1	(CalcHandle* handle)
 {
-	TRYF(rd_send(handle, "romdump.8Xp", romDumpSize84p, romDump84p));
+	TRYF(rd_send(handle, "romdump-usb.8Xp", romDumpSize84p, romDump84p));
 
 	return 0;
 }
--- a/src/calc_89t.c
+++ b/src/calc_89t.c
@@ -566,7 +566,7 @@
 	dusb_cp_del(param);
 
 	// Send dumping program
-	TRYF(rd_send(handle, "romdump.89z", romDumpSize89t, romDump89t));
+	TRYF(rd_send(handle, "romdump-usb.89z", romDumpSize89t, romDump89t));
 	PAUSE(1000);
 
 	return 0;
--- a/src/error.h
+++ b/src/error.h
@@ -65,6 +65,8 @@
 	ERR_INVALID_HANDLE,		// Invalid handle
 	ERR_INVALID_PARAMETER,
 
+    ERR_ROMDUMP_UNINSTALLED,
+
 // --- 300 to 349 are reserved for hand-held status (DUSB)
 
 	ERR_CALC_ERROR2 = 300,	// Hand-held returned an error code
--- a/src/romdump.c
+++ b/src/romdump.c
@@ -377,6 +377,7 @@
 
 int rd_send(CalcHandle *h, const char *prgname, uint16_t size, uint8_t *data)
 {
+#if 0
 	char *template, *tempfname;
 	int fd, ret;
 
@@ -406,4 +407,26 @@
 	g_free(tempfname);
 
 	return ret;
+#else
+	char *filename;
+	int ret;
+
+	// Transfer program to calc
+	h->busy = 0;
+	filename = g_build_filename("/usr/share/libticalcs2", prgname, NULL);
+
+	if (!g_file_test(filename, G_FILE_TEST_IS_REGULAR))
+	{
+		g_free(filename);
+		return ERR_ROMDUMP_UNINSTALLED;
+	}
+
+	ret = ticalcs_calc_send_var2(h, MODE_NORMAL, filename);
+	g_free(filename);
+
+	(void) size;
+	(void) data;
+
+	return ret;
+#endif
 }
--- a/src/rom73.h
+++ b/src/rom73.h
@@ -1,107 +1,3 @@
 static unsigned char romDump73[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x37, 0x33, 0x2a, 0x2a, 
-0x1a, 0x0a, 0x00, 0x43, 0x72, 0x65, 0x61, 0x74, 
-0x65, 0x64, 0x20, 0x62, 0x79, 0x20, 0x74, 0x69, 
-0x70, 0x61, 0x63, 0x6b, 0x20, 0x30, 0x2e, 0x31, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0x0b, 
-0x00, 0xf1, 0x02, 0x06, 0x52, 0x4f, 0x4d, 0x44, 
-0x55, 0x4d, 0x50, 0x00, 0xf1, 0x02, 0xef, 0x02, 
-0x5f, 0x30, 0x30, 0x30, 0x30, 0x31, 0x30, 0x30, 
-0x35, 0x3f, 0x2a, 0x5d, 0x9a, 0xe9, 0x01, 0xd5, 
-0x3f, 0x11, 0x2b, 0x01, 0x19, 0x11, 0x52, 0x8a, 
-0x01, 0xb6, 0x01, 0xed, 0xb0, 0xfd, 0xcb, 0x1f, 
-0xae, 0xfd, 0xcb, 0x08, 0x9e, 0xdb, 0x06, 0x32, 
-0x13, 0x86, 0xcd, 0x53, 0x8a, 0x3a, 0x13, 0x86, 
-0xd3, 0x06, 0x21, 0x00, 0x00, 0x22, 0x16, 0x86, 
-0x22, 0x18, 0x86, 0xef, 0xd7, 0x44, 0xef, 0xad, 
-0x44, 0x21, 0x00, 0x00, 0x22, 0xe2, 0x83, 0x21, 
-0xd2, 0x8b, 0xef, 0x71, 0x44, 0x11, 0x04, 0x00, 
-0xed, 0x53, 0xe2, 0x83, 0xef, 0x71, 0x44, 0xfb, 
-0x76, 0xef, 0x0f, 0x40, 0xfe, 0x37, 0x28, 0x2f, 
-0xcd, 0x6a, 0x8b, 0x28, 0xf2, 0xcd, 0xca, 0x8a, 
-0x38, 0x53, 0xed, 0x5b, 0x26, 0x86, 0x21, 0x55, 
-0xaa, 0xb7, 0xed, 0x52, 0x28, 0x09, 0x7a, 0xb7, 
-0x20, 0x43, 0x43, 0x10, 0x0a, 0x18, 0xd8, 0x21, 
-0x01, 0x00, 0xcd, 0x00, 0x8b, 0x18, 0xd0, 0x10, 
-0x10, 0x21, 0x02, 0x00, 0xcd, 0x00, 0x8b, 0xef, 
-0xad, 0x44, 0x21, 0x00, 0x00, 0x22, 0xe2, 0x83, 
-0xc9, 0x10, 0x20, 0x21, 0x04, 0x8c, 0x22, 0x1e, 
-0x86, 0x21, 0x04, 0x00, 0x22, 0x1c, 0x86, 0x21, 
-0x03, 0x00, 0xcd, 0x08, 0x8b, 0x21, 0x02, 0x04, 
-0x22, 0xe2, 0x83, 0x21, 0xfa, 0x8b, 0xef, 0x71, 
-0x44, 0x18, 0x9c, 0x10, 0x1b, 0x21, 0x07, 0x00, 
-0x22, 0xe2, 0x83, 0x21, 0xf2, 0x8b, 0xef, 0x71, 
-0x44, 0x2a, 0x16, 0x86, 0x23, 0x22, 0x16, 0x86, 
-0xef, 0x6e, 0x44, 0xcd, 0x5c, 0x8b, 0x18, 0xe1, 
-0x10, 0xe3, 0x2a, 0x2a, 0x86, 0x3a, 0x2c, 0x86, 
-0xcb, 0x24, 0x17, 0xcb, 0x24, 0x17, 0xd3, 0x06, 
-0x37, 0xcb, 0x1c, 0xcb, 0x3c, 0x22, 0x1e, 0x86, 
-0xcd, 0xc0, 0x8a, 0x20, 0x19, 0x67, 0x6f, 0x22, 
-0x22, 0x86, 0x21, 0x00, 0x04, 0x22, 0x20, 0x86, 
-0x21, 0x20, 0x86, 0x22, 0x1e, 0x86, 0x21, 0x04, 
-0x00, 0x11, 0x07, 0x00, 0x18, 0x06, 0x21, 0x00, 
-0x04, 0x11, 0x06, 0x00, 0x22, 0x1c, 0x86, 0xeb, 
-0xcd, 0x08, 0x8b, 0x3a, 0x13, 0x86, 0xd3, 0x06, 
-0x38, 0x9b, 0x21, 0x02, 0x00, 0x22, 0xe2, 0x83, 
-0x2a, 0x18, 0x86, 0x23, 0x22, 0x18, 0x86, 0xef, 
-0x6e, 0x44, 0x3e, 0x4b, 0xef, 0x6b, 0x44, 0x18, 
-0x9d, 0xe9, 0xef, 0x8e, 0x4b, 0x48, 0x47, 0x21, 
-0x91, 0x01, 0xb7, 0xed, 0x42, 0xd8, 0xf3, 0x3e, 
-0x1c, 0xd3, 0x06, 0x21, 0x00, 0x40, 0x44, 0x4d, 
-0x3e, 0xf5, 0xed, 0xb1, 0x37, 0xc0, 0xe5, 0xc5, 
-0x11, 0xb0, 0x8a, 0x06, 0x0c, 0x1a, 0xbe, 0x20, 
-0x0c, 0x23, 0x13, 0x10, 0xf8, 0x23, 0x23, 0x23, 
-0x87, 0x06, 0x04, 0x20, 0xf0, 0xc1, 0xe1, 0x20, 
-0xdf, 0xed, 0x73, 0x13, 0x86, 0x2b, 0xe5, 0x21, 
-0xaa, 0x8a, 0x22, 0x15, 0x86, 0x21, 0x15, 0x86, 
-0xe5, 0x11, 0x17, 0x86, 0xef, 0xfc, 0x46, 0x21, 
-0xf0, 0xff, 0x39, 0xeb, 0xe1, 0x01, 0x14, 0x00, 
-0xc9, 0xed, 0x7b, 0x13, 0x86, 0xaf, 0xc9, 0x3e, 
-0x01, 0x00, 0xf3, 0x00, 0x00, 0xed, 0x56, 0xf3, 
-0xd3, 0x14, 0xf3, 0xf1, 0xef, 0x54, 0x80, 0x7e, 
-0x01, 0x00, 0x04, 0xed, 0xa1, 0xe0, 0x28, 0xfb, 
-0xc9, 0x21, 0x00, 0x00, 0x11, 0x26, 0x86, 0x06, 
-0x04, 0xcd, 0x45, 0x8b, 0xd8, 0x3a, 0x29, 0x86, 
-0xc6, 0xff, 0xd8, 0x3a, 0x28, 0x86, 0xb7, 0x28, 
-0x0b, 0x47, 0xc6, 0xbf, 0xd8, 0x11, 0x2a, 0x86, 
-0xcd, 0x45, 0x8b, 0xd8, 0xe5, 0x11, 0x24, 0x86, 
-0x06, 0x02, 0xcd, 0x45, 0x8b, 0xd1, 0xd8, 0x2a, 
-0x24, 0x86, 0xed, 0x52, 0xc8, 0x37, 0xc9, 0xe5, 
-0x21, 0x00, 0x00, 0x22, 0x1c, 0x86, 0xe1, 0x22, 
-0x1a, 0x86, 0x21, 0x00, 0x00, 0x11, 0x1a, 0x86, 
-0x01, 0x04, 0x00, 0xcd, 0x2f, 0x8b, 0xd8, 0xed, 
-0x5b, 0x1e, 0x86, 0xed, 0x4b, 0x1c, 0x86, 0xcd, 
-0x2f, 0x8b, 0xd8, 0x22, 0x24, 0x86, 0x11, 0x24, 
-0x86, 0x01, 0x02, 0x00, 0x18, 0x00, 0x78, 0xb1, 
-0xc8, 0x0b, 0xc5, 0x1a, 0x13, 0x4f, 0x06, 0x00, 
-0x09, 0xd5, 0xe5, 0xcd, 0xa6, 0x8b, 0xe1, 0xd1, 
-0xc1, 0xd8, 0x18, 0xea, 0xc5, 0xe5, 0xd5, 0xcd, 
-0x70, 0x8b, 0xd1, 0xe1, 0x38, 0x0b, 0x06, 0x00, 
-0x09, 0x79, 0x12, 0x13, 0xc1, 0x10, 0xed, 0xb7, 
-0xc5, 0xc1, 0xc9, 0x06, 0xff, 0xf5, 0x3e, 0x00, 
-0xd3, 0x00, 0xfb, 0x76, 0x76, 0x10, 0xfb, 0xf1, 
-0xc9, 0xdb, 0x00, 0x2f, 0xe6, 0x03, 0xc9, 0xf3, 
-0x3e, 0x00, 0xd3, 0x00, 0x06, 0x08, 0x16, 0xff, 
-0xcd, 0x96, 0x8b, 0xd8, 0xc6, 0xfe, 0x3e, 0x02, 
-0x30, 0x03, 0xc0, 0x3e, 0x01, 0xd3, 0x00, 0xcb, 
-0x19, 0x14, 0xcd, 0x96, 0x8b, 0xd8, 0x3e, 0x00, 
-0xd3, 0x00, 0x10, 0xe2, 0xc9, 0x21, 0xff, 0xff, 
-0x2b, 0xdb, 0x00, 0xaa, 0xe6, 0x03, 0xc0, 0x7c, 
-0xb5, 0x20, 0xf5, 0x37, 0xc9, 0xf3, 0x06, 0x08, 
-0xcd, 0xbc, 0x8b, 0xd8, 0xcb, 0x19, 0x3e, 0x01, 
-0x30, 0x02, 0x3e, 0x02, 0x14, 0xcd, 0xc0, 0x8b, 
-0xd8, 0x10, 0xed, 0x3e, 0x00, 0x16, 0xff, 0xd3, 
-0x00, 0x21, 0xff, 0xff, 0x2b, 0xdb, 0x00, 0xaa, 
-0xe6, 0x03, 0xc8, 0x7c, 0xb5, 0x20, 0xf5, 0x37, 
-0xc9, 0x44, 0x75, 0x6d, 0x70, 0x69, 0x6e, 0x67, 
-0x20, 0x52, 0x4f, 0x4d, 0x2e, 0x2e, 0x2e, 0x00, 
-0xc1, 0x4d, 0x4f, 0x44, 0x45, 0x5d, 0x20, 0x74, 
-0x6f, 0x20, 0x63, 0x61, 0x6e, 0x63, 0x65, 0x6c, 
-0x00, 0x45, 0x72, 0x72, 0x6f, 0x72, 0x73, 0x3a, 
-0x00, 0x30, 0x4b, 0x20, 0x6f, 0x66, 0x20, 0x35, 
-0x31, 0x32, 0x4b, 0x00, 0x00, 0x08, 0x00, 0x2e, 
-0x2d, 
 };
-static unsigned int romDumpSize73 = 825;
+static unsigned int romDumpSize73 = 0;
--- a/src/rom82.h
+++ b/src/rom82.h
@@ -1,161 +1,3 @@
 static unsigned char romDump82[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x38, 0x32, 0x2a, 0x2a, 
-0x1a, 0x0a, 0x00, 0x43, 0x72, 0x65, 0x61, 0x74, 
-0x65, 0x64, 0x20, 0x62, 0x79, 0x20, 0x74, 0x69, 
-0x70, 0x61, 0x63, 0x6b, 0x20, 0x30, 0x2e, 0x31, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0xb4, 0x04, 0x0b, 
-0x00, 0xa5, 0x04, 0x03, 0x24, 0x52, 0x44, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0xa5, 0x04, 0xa3, 0x04, 
-0x11, 0x65, 0x01, 0x19, 0x11, 0xa8, 0x82, 0x01, 
-0x06, 0x02, 0xed, 0xb0, 0x22, 0x34, 0x84, 0xed, 
-0x73, 0x1a, 0x82, 0x11, 0xde, 0xfc, 0x19, 0xcd, 
-0xa8, 0x82, 0x2a, 0x10, 0x8d, 0x22, 0x12, 0x8d, 
-0x2a, 0x0c, 0x8d, 0x22, 0x0e, 0x8d, 0x21, 0x08, 
-0x00, 0x39, 0x5e, 0x23, 0x56, 0x21, 0x06, 0x00, 
-0x19, 0x7e, 0x23, 0xae, 0x23, 0xae, 0xfe, 0xa1, 
-0x20, 0x03, 0x3e, 0x40, 0xe9, 0xe1, 0xe1, 0xe1, 
-0xf1, 0xd3, 0x02, 0x3e, 0x06, 0x32, 0x19, 0x82, 
-0xc9, 0xfd, 0xcb, 0x08, 0x9e, 0xdb, 0x02, 0x32, 
-0x28, 0x82, 0xcd, 0x53, 0x83, 0x21, 0x00, 0x00, 
-0x22, 0x2b, 0x82, 0x22, 0x2d, 0x82, 0xfd, 0xcb, 
-0x12, 0x86, 0xcd, 0x19, 0x84, 0x21, 0x00, 0x00, 
-0x22, 0x0c, 0x80, 0x21, 0x78, 0x84, 0xcd, 0x10, 
-0x84, 0x11, 0x04, 0x00, 0xed, 0x53, 0x0c, 0x80, 
-0xcd, 0x10, 0x84, 0xfb, 0x76, 0xf3, 0x21, 0x00, 
-0x80, 0x7e, 0x36, 0x00, 0xfe, 0x37, 0x28, 0x2f, 
-0xcd, 0x76, 0x83, 0x28, 0xee, 0xcd, 0xb3, 0x82, 
-0x38, 0x53, 0xed, 0x5b, 0x3b, 0x82, 0x21, 0x55, 
-0xaa, 0xb7, 0xed, 0x52, 0x28, 0x09, 0x7a, 0xb7, 
-0x20, 0x43, 0x43, 0x10, 0x0a, 0x18, 0xd4, 0x21, 
-0x01, 0x00, 0xcd, 0xe9, 0x82, 0x18, 0xcc, 0x10, 
-0x10, 0x21, 0x02, 0x00, 0xcd, 0xe9, 0x82, 0xcd, 
-0x19, 0x84, 0x21, 0x00, 0x00, 0x22, 0x0c, 0x80, 
-0xc9, 0x10, 0x20, 0x21, 0xaa, 0x84, 0x22, 0x33, 
-0x82, 0x21, 0x04, 0x00, 0x22, 0x31, 0x82, 0x21, 
-0x03, 0x00, 0xcd, 0xf1, 0x82, 0x21, 0x02, 0x04, 
-0x22, 0x0c, 0x80, 0x21, 0xa0, 0x84, 0xcd, 0x10, 
-0x84, 0x18, 0x98, 0x10, 0x1b, 0x21, 0x07, 0x00, 
-0x22, 0x0c, 0x80, 0x21, 0x98, 0x84, 0xcd, 0x10, 
-0x84, 0x2a, 0x2b, 0x82, 0x23, 0x22, 0x2b, 0x82, 
-0xcd, 0xe4, 0x83, 0xcd, 0x45, 0x83, 0x18, 0xe1, 
-0x10, 0xe3, 0x2a, 0x3f, 0x82, 0x3a, 0x41, 0x82, 
-0xcb, 0x24, 0x17, 0xcb, 0x24, 0x17, 0xe6, 0x07, 
-0xf6, 0x88, 0xd3, 0x02, 0x37, 0xcb, 0x1c, 0xcb, 
-0x3c, 0x22, 0x33, 0x82, 0xcd, 0xa9, 0x82, 0x20, 
-0x19, 0x67, 0x6f, 0x22, 0x37, 0x82, 0x21, 0x00, 
-0x04, 0x22, 0x35, 0x82, 0x21, 0x35, 0x82, 0x22, 
-0x33, 0x82, 0x21, 0x04, 0x00, 0x11, 0x07, 0x00, 
-0x18, 0x06, 0x21, 0x00, 0x04, 0x11, 0x06, 0x00, 
-0x22, 0x31, 0x82, 0xeb, 0xcd, 0xf1, 0x82, 0x3a, 
-0x28, 0x82, 0xd3, 0x02, 0x38, 0x97, 0x21, 0x02, 
-0x00, 0x22, 0x0c, 0x80, 0x2a, 0x2d, 0x82, 0x23, 
-0x22, 0x2d, 0x82, 0xcd, 0xe4, 0x83, 0x3e, 0x4b, 
-0xcd, 0x30, 0x84, 0x18, 0x99, 0xe9, 0x7e, 0x01, 
-0x00, 0x04, 0xed, 0xa1, 0xe0, 0x28, 0xfb, 0xc9, 
-0x21, 0x00, 0x00, 0x11, 0x3b, 0x82, 0x06, 0x04, 
-0xcd, 0x2e, 0x83, 0xd8, 0x3a, 0x3e, 0x82, 0xc6, 
-0xff, 0xd8, 0x3a, 0x3d, 0x82, 0xb7, 0x28, 0x0b, 
-0x47, 0xc6, 0xbf, 0xd8, 0x11, 0x3f, 0x82, 0xcd, 
-0x2e, 0x83, 0xd8, 0xe5, 0x11, 0x39, 0x82, 0x06, 
-0x02, 0xcd, 0x2e, 0x83, 0xd1, 0xd8, 0x2a, 0x39, 
-0x82, 0xed, 0x52, 0xc8, 0x37, 0xc9, 0xe5, 0x21, 
-0x00, 0x00, 0x22, 0x31, 0x82, 0xe1, 0x22, 0x2f, 
-0x82, 0x21, 0x00, 0x00, 0x11, 0x2f, 0x82, 0x01, 
-0x04, 0x00, 0xcd, 0x18, 0x83, 0xd8, 0xed, 0x5b, 
-0x33, 0x82, 0xed, 0x4b, 0x31, 0x82, 0xcd, 0x18, 
-0x83, 0xd8, 0x22, 0x39, 0x82, 0x11, 0x39, 0x82, 
-0x01, 0x02, 0x00, 0x18, 0x00, 0x78, 0xb1, 0xc8, 
-0x0b, 0xc5, 0x1a, 0x13, 0x4f, 0x06, 0x00, 0x09, 
-0xd5, 0xe5, 0xcd, 0xb5, 0x83, 0xe1, 0xd1, 0xc1, 
-0xd8, 0x18, 0xea, 0xc5, 0xe5, 0xd5, 0xcd, 0x7c, 
-0x83, 0xd1, 0xe1, 0x38, 0x0b, 0x06, 0x00, 0x09, 
-0x79, 0x12, 0x13, 0xc1, 0x10, 0xed, 0xb7, 0xc5, 
-0xc1, 0xc9, 0x06, 0xff, 0xf5, 0x3e, 0xc0, 0xd3, 
-0x00, 0xfb, 0x76, 0x76, 0x10, 0xfb, 0xf1, 0xc9, 
-0xf3, 0x3e, 0xc0, 0xd3, 0x00, 0x01, 0x03, 0xfe, 
-0x21, 0xd4, 0xe8, 0xdb, 0x00, 0xe6, 0x0c, 0x28, 
-0x06, 0x01, 0x0c, 0xf8, 0x21, 0xc1, 0xc2, 0x79, 
-0x32, 0x7a, 0x83, 0x78, 0x32, 0x8a, 0x83, 0x22, 
-0x29, 0x82, 0xc9, 0xdb, 0x00, 0x2f, 0xe6, 0x00, 
-0xc9, 0xf3, 0x3e, 0xc0, 0xd3, 0x00, 0x06, 0x08, 
-0x16, 0xff, 0xcd, 0xa4, 0x83, 0xd8, 0xc6, 0x00, 
-0x3a, 0x2a, 0x82, 0x30, 0x04, 0xc0, 0x3a, 0x29, 
-0x82, 0xd3, 0x00, 0xcb, 0x19, 0x14, 0xcd, 0xa4, 
-0x83, 0xd8, 0x3e, 0xc0, 0xd3, 0x00, 0x10, 0xe0, 
-0xc9, 0x21, 0xff, 0xff, 0x2b, 0xdb, 0x00, 0xaa, 
-0xcd, 0x79, 0x83, 0xc0, 0x7c, 0xb5, 0x20, 0xf4, 
-0x37, 0xc9, 0xf3, 0x06, 0x08, 0xcd, 0xcd, 0x83, 
-0xd8, 0xcb, 0x19, 0x3a, 0x29, 0x82, 0x30, 0x03, 
-0x3a, 0x2a, 0x82, 0x14, 0xcd, 0xd1, 0x83, 0xd8, 
-0x10, 0xeb, 0x3e, 0xc0, 0x16, 0xff, 0xd3, 0x00, 
-0x21, 0xff, 0xff, 0x2b, 0xdb, 0x00, 0xaa, 0xcd, 
-0x79, 0x83, 0xc8, 0x7c, 0xb5, 0x20, 0xf4, 0x37, 
-0xc9, 0x06, 0x05, 0xeb, 0x3e, 0x20, 0xcd, 0x30, 
-0x84, 0x10, 0xf9, 0x21, 0x0d, 0x80, 0x7e, 0xf5, 
-0x35, 0xeb, 0x01, 0x0a, 0x10, 0xaf, 0x29, 0x8f, 
-0xb9, 0x38, 0x02, 0x91, 0x2c, 0x10, 0xf7, 0xeb, 
-0xc6, 0x30, 0xcd, 0x30, 0x84, 0x35, 0x7a, 0xb3, 
-0x20, 0xe6, 0xf1, 0x77, 0xc9, 0x7e, 0x23, 0xb7, 
-0xc8, 0xcd, 0x30, 0x84, 0x18, 0xf7, 0x3e, 0x20, 
-0x4f, 0xcd, 0x6a, 0x84, 0x06, 0x40, 0xcd, 0x72, 
-0x84, 0xaf, 0xd3, 0x11, 0x10, 0xf8, 0x79, 0x3c, 
-0xfe, 0x2c, 0x38, 0xec, 0xc9, 0xe5, 0xd5, 0xc5, 
-0x21, 0x00, 0x00, 0x01, 0x08, 0x00, 0xbe, 0x28, 
-0x03, 0x09, 0x30, 0xfa, 0x23, 0xaf, 0xcd, 0x6a, 
-0x84, 0x11, 0x0d, 0x80, 0x1a, 0x3c, 0x12, 0x3d, 
-0xf6, 0x20, 0xcd, 0x6a, 0x84, 0x1b, 0x1a, 0x87, 
-0x87, 0x87, 0xf6, 0x80, 0xcd, 0x6a, 0x84, 0x06, 
-0x07, 0xcd, 0x72, 0x84, 0x7e, 0xd3, 0x11, 0x23, 
-0x10, 0xf7, 0xc1, 0xd1, 0xe1, 0x3e, 0x01, 0xf5, 
-0xcd, 0x72, 0x84, 0xf1, 0xd3, 0x10, 0xc9, 0xdb, 
-0x10, 0x87, 0x38, 0xfb, 0xc9, 0x44, 0x75, 0x6d, 
-0x70, 0x69, 0x6e, 0x67, 0x20, 0x52, 0x4f, 0x4d, 
-0x2e, 0x2e, 0x2e, 0x00, 0x5b, 0x4d, 0x4f, 0x44, 
-0x45, 0x5d, 0x20, 0x74, 0x6f, 0x20, 0x63, 0x61, 
-0x6e, 0x63, 0x65, 0x6c, 0x00, 0x45, 0x72, 0x72, 
-0x6f, 0x72, 0x73, 0x3a, 0x00, 0x30, 0x4b, 0x20, 
-0x6f, 0x66, 0x20, 0x31, 0x32, 0x38, 0x4b, 0x00, 
-0x00, 0x02, 0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x2e, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x0c, 0x0c, 0x3a, 0x00, 0x0c, 0x0c, 0x00, 
-0x0c, 0x0c, 0x00, 0x5b, 0x0c, 0x08, 0x08, 0x08, 
-0x08, 0x08, 0x0c, 0x5d, 0x18, 0x08, 0x08, 0x08, 
-0x08, 0x08, 0x18, 0x30, 0x1c, 0x22, 0x26, 0x2a, 
-0x32, 0x22, 0x1c, 0x31, 0x08, 0x18, 0x08, 0x08, 
-0x08, 0x08, 0x1c, 0x32, 0x1c, 0x22, 0x02, 0x04, 
-0x08, 0x10, 0x3e, 0x33, 0x3e, 0x04, 0x08, 0x04, 
-0x02, 0x22, 0x1c, 0x34, 0x04, 0x0c, 0x14, 0x24, 
-0x3e, 0x04, 0x04, 0x35, 0x3e, 0x20, 0x3c, 0x02, 
-0x02, 0x22, 0x1c, 0x36, 0x0c, 0x10, 0x20, 0x3c, 
-0x22, 0x22, 0x1c, 0x37, 0x3e, 0x02, 0x04, 0x08, 
-0x10, 0x10, 0x10, 0x38, 0x1c, 0x22, 0x22, 0x1c, 
-0x22, 0x22, 0x1c, 0x39, 0x1c, 0x22, 0x22, 0x1e, 
-0x02, 0x04, 0x18, 0x44, 0x3c, 0x22, 0x22, 0x22, 
-0x22, 0x22, 0x3c, 0x45, 0x3e, 0x20, 0x20, 0x3c, 
-0x20, 0x20, 0x3e, 0x49, 0x1c, 0x08, 0x08, 0x08, 
-0x08, 0x08, 0x1c, 0x4b, 0x22, 0x24, 0x28, 0x30, 
-0x28, 0x24, 0x22, 0x4d, 0x22, 0x36, 0x2a, 0x2a, 
-0x22, 0x22, 0x22, 0x4f, 0x1c, 0x22, 0x22, 0x22, 
-0x22, 0x22, 0x1c, 0x52, 0x3c, 0x22, 0x22, 0x3c, 
-0x28, 0x24, 0x22, 0x54, 0x3e, 0x08, 0x08, 0x08, 
-0x08, 0x08, 0x08, 0x58, 0x22, 0x22, 0x14, 0x08, 
-0x14, 0x22, 0x22, 0x61, 0x00, 0x00, 0x1c, 0x02, 
-0x1e, 0x22, 0x1e, 0x63, 0x00, 0x00, 0x1c, 0x20, 
-0x20, 0x22, 0x1c, 0x65, 0x00, 0x00, 0x1c, 0x22, 
-0x3e, 0x20, 0x1c, 0x66, 0x0c, 0x12, 0x10, 0x38, 
-0x10, 0x10, 0x10, 0x67, 0x00, 0x1e, 0x22, 0x22, 
-0x1e, 0x02, 0x1c, 0x69, 0x08, 0x00, 0x18, 0x08, 
-0x08, 0x08, 0x1c, 0x6c, 0x18, 0x08, 0x08, 0x08, 
-0x08, 0x08, 0x1c, 0x6d, 0x00, 0x00, 0x34, 0x2a, 
-0x2a, 0x22, 0x22, 0x6e, 0x00, 0x00, 0x2c, 0x32, 
-0x22, 0x22, 0x22, 0x6f, 0x00, 0x00, 0x1c, 0x22, 
-0x22, 0x22, 0x1c, 0x70, 0x00, 0x00, 0x3c, 0x22, 
-0x3c, 0x20, 0x20, 0x72, 0x00, 0x00, 0x2c, 0x32, 
-0x20, 0x20, 0x20, 0x73, 0x00, 0x00, 0x1c, 0x20, 
-0x1c, 0x02, 0x3c, 0x74, 0x10, 0x10, 0x38, 0x10, 
-0x10, 0x12, 0x0c, 0x75, 0x00, 0x00, 0x22, 0x22, 
-0x22, 0x26, 0x1a, 0xde, 0x86, 
 };
-static unsigned int romDumpSize82 = 1261;
+static unsigned int romDumpSize82 = 0;
--- a/src/rom83.h
+++ b/src/rom83.h
@@ -1,100 +1,3 @@
 static unsigned char romDump83[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x38, 0x33, 0x2a, 0x2a, 
-0x1a, 0x0a, 0x00, 0x43, 0x72, 0x65, 0x61, 0x74, 
-0x65, 0x64, 0x20, 0x62, 0x79, 0x20, 0x74, 0x69, 
-0x70, 0x61, 0x63, 0x6b, 0x20, 0x30, 0x2e, 0x31, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0xca, 0x02, 0x0b, 
-0x00, 0xbb, 0x02, 0x06, 0x52, 0x4f, 0x4d, 0x44, 
-0x55, 0x4d, 0x50, 0x00, 0xbb, 0x02, 0xb9, 0x02, 
-0x43, 0x44, 0x32, 0x36, 0x34, 0x34, 0x32, 0x31, 
-0x31, 0x39, 0x30, 0x30, 0x31, 0x39, 0x45, 0x39, 
-0xd4, 0x3f, 0x30, 0x30, 0x30, 0x30, 0x3f, 0x11, 
-0x37, 0x01, 0x19, 0x11, 0xe5, 0x82, 0x01, 0x63, 
-0x01, 0xed, 0xb0, 0xfd, 0xcb, 0x08, 0x9e, 0xdb, 
-0x02, 0x32, 0x65, 0x82, 0xdb, 0x00, 0xe6, 0x10, 
-0xf6, 0xc0, 0x32, 0x66, 0x82, 0x21, 0x00, 0x00, 
-0x22, 0x68, 0x82, 0x22, 0x6a, 0x82, 0xcd, 0x95, 
-0x47, 0xcd, 0x5d, 0x47, 0x21, 0x00, 0x00, 0x22, 
-0x0c, 0x80, 0x21, 0x12, 0x84, 0xcd, 0x0d, 0x47, 
-0x11, 0x04, 0x00, 0xed, 0x53, 0x0c, 0x80, 0xcd, 
-0x0d, 0x47, 0xfb, 0x76, 0xcd, 0x14, 0x40, 0xfe, 
-0x37, 0x28, 0x2f, 0xcd, 0xaa, 0x83, 0x28, 0xf2, 
-0xcd, 0xf0, 0x82, 0x38, 0x53, 0xed, 0x5b, 0x78, 
-0x82, 0x21, 0x55, 0xaa, 0xb7, 0xed, 0x52, 0x28, 
-0x09, 0x7a, 0xb7, 0x20, 0x43, 0x43, 0x10, 0x0a, 
-0x18, 0xd8, 0x21, 0x01, 0x00, 0xcd, 0x26, 0x83, 
-0x18, 0xd0, 0x10, 0x10, 0x21, 0x02, 0x00, 0xcd, 
-0x26, 0x83, 0xcd, 0x5d, 0x47, 0x21, 0x00, 0x00, 
-0x22, 0x0c, 0x80, 0xc9, 0x10, 0x20, 0x21, 0x44, 
-0x84, 0x22, 0x70, 0x82, 0x21, 0x04, 0x00, 0x22, 
-0x6e, 0x82, 0x21, 0x03, 0x00, 0xcd, 0x2e, 0x83, 
-0x21, 0x02, 0x04, 0x22, 0x0c, 0x80, 0x21, 0x3a, 
-0x84, 0xcd, 0x0d, 0x47, 0x18, 0x9c, 0x10, 0x1b, 
-0x21, 0x07, 0x00, 0x22, 0x0c, 0x80, 0x21, 0x32, 
-0x84, 0xcd, 0x0d, 0x47, 0x2a, 0x68, 0x82, 0x23, 
-0x22, 0x68, 0x82, 0xcd, 0x09, 0x47, 0xcd, 0x98, 
-0x83, 0x18, 0xe1, 0x10, 0xe3, 0x2a, 0x7c, 0x82, 
-0x3a, 0x7e, 0x82, 0xcb, 0x24, 0x17, 0xcb, 0x24, 
-0x17, 0xf5, 0xe6, 0x08, 0x07, 0xf6, 0xc0, 0xd3, 
-0x00, 0xf1, 0xe6, 0x07, 0xf6, 0x88, 0xd3, 0x02, 
-0x37, 0xcb, 0x1c, 0xcb, 0x3c, 0x22, 0x70, 0x82, 
-0xcd, 0xe6, 0x82, 0x20, 0x19, 0x67, 0x6f, 0x22, 
-0x74, 0x82, 0x21, 0x00, 0x04, 0x22, 0x72, 0x82, 
-0x21, 0x72, 0x82, 0x22, 0x70, 0x82, 0x21, 0x04, 
-0x00, 0x11, 0x07, 0x00, 0x18, 0x06, 0x21, 0x00, 
-0x04, 0x11, 0x06, 0x00, 0x22, 0x6e, 0x82, 0xeb, 
-0xcd, 0x2e, 0x83, 0x3a, 0x66, 0x82, 0xd3, 0x00, 
-0x3a, 0x65, 0x82, 0xd3, 0x02, 0x38, 0x89, 0x21, 
-0x02, 0x00, 0x22, 0x0c, 0x80, 0x2a, 0x6a, 0x82, 
-0x23, 0x22, 0x6a, 0x82, 0xcd, 0x09, 0x47, 0x3e, 
-0x4b, 0xcd, 0x05, 0x47, 0x18, 0x8b, 0xe9, 0x7e, 
-0x01, 0x00, 0x04, 0xed, 0xa1, 0xe0, 0x28, 0xfb, 
-0xc9, 0x21, 0x00, 0x00, 0x11, 0x78, 0x82, 0x06, 
-0x04, 0xcd, 0x77, 0x83, 0xd8, 0x3a, 0x7b, 0x82, 
-0xc6, 0xff, 0xd8, 0x3a, 0x7a, 0x82, 0xb7, 0x28, 
-0x0b, 0x47, 0xc6, 0xbf, 0xd8, 0x11, 0x7c, 0x82, 
-0xcd, 0x77, 0x83, 0xd8, 0xe5, 0x11, 0x76, 0x82, 
-0x06, 0x02, 0xcd, 0x77, 0x83, 0xd1, 0xd8, 0x2a, 
-0x76, 0x82, 0xed, 0x52, 0xc8, 0x37, 0xc9, 0xe5, 
-0x21, 0x00, 0x00, 0x22, 0x6e, 0x82, 0xe1, 0x22, 
-0x6c, 0x82, 0x21, 0x00, 0x00, 0x11, 0x6c, 0x82, 
-0x01, 0x04, 0x00, 0xcd, 0x55, 0x83, 0xd8, 0xed, 
-0x5b, 0x70, 0x82, 0xed, 0x4b, 0x6e, 0x82, 0xcd, 
-0x55, 0x83, 0xd8, 0x22, 0x76, 0x82, 0x11, 0x76, 
-0x82, 0x01, 0x02, 0x00, 0x18, 0x00, 0x78, 0xb1, 
-0xc8, 0x0b, 0xc5, 0xdb, 0x00, 0xe6, 0x10, 0xf6, 
-0xc0, 0xf5, 0x1a, 0x13, 0x4f, 0x06, 0x00, 0x09, 
-0xd5, 0xe5, 0xcd, 0xe6, 0x83, 0xe1, 0xd1, 0xc1, 
-0x0e, 0x00, 0xed, 0x41, 0xc1, 0xd8, 0x18, 0xde, 
-0xdb, 0x00, 0xe6, 0x10, 0xf6, 0xc0, 0x4f, 0xc5, 
-0xe5, 0xd5, 0xcd, 0xb0, 0x83, 0xd1, 0xe1, 0x38, 
-0x0b, 0x06, 0x00, 0x09, 0x79, 0x12, 0x13, 0xc1, 
-0x10, 0xed, 0xb7, 0xc5, 0xc1, 0x79, 0xd3, 0x00, 
-0xc9, 0x06, 0xff, 0xf5, 0xdb, 0x00, 0xe6, 0x10, 
-0xf6, 0xc0, 0xd3, 0x00, 0xfb, 0x76, 0x76, 0x10, 
-0xfb, 0xf1, 0xc9, 0xdb, 0x00, 0x2f, 0xe6, 0x0c, 
-0xc9, 0xf3, 0x3e, 0xc0, 0xd3, 0x00, 0x06, 0x08, 
-0x16, 0xff, 0xcd, 0xd6, 0x83, 0xd8, 0xc6, 0xf8, 
-0x3e, 0xc2, 0x30, 0x03, 0xc0, 0x3e, 0xc1, 0xd3, 
-0x00, 0xcb, 0x19, 0x14, 0xcd, 0xd6, 0x83, 0xd8, 
-0x3e, 0xc0, 0xd3, 0x00, 0x10, 0xe2, 0xc9, 0x21, 
-0xff, 0xff, 0x2b, 0xdb, 0x00, 0xaa, 0xe6, 0x0c, 
-0xc0, 0x7c, 0xb5, 0x20, 0xf5, 0x37, 0xc9, 0xf3, 
-0x06, 0x08, 0xcd, 0xfc, 0x83, 0xd8, 0xcb, 0x19, 
-0x3e, 0xc1, 0x30, 0x02, 0x3e, 0xc2, 0x14, 0xcd, 
-0x00, 0x84, 0xd8, 0x10, 0xed, 0x3e, 0xc0, 0x16, 
-0xff, 0xd3, 0x00, 0x21, 0xff, 0xff, 0x2b, 0xdb, 
-0x00, 0xaa, 0xe6, 0x0c, 0xc8, 0x7c, 0xb5, 0x20, 
-0xf5, 0x37, 0xc9, 0x44, 0x75, 0x6d, 0x70, 0x69, 
-0x6e, 0x67, 0x20, 0x52, 0x4f, 0x4d, 0x2e, 0x2e, 
-0x2e, 0x00, 0xc1, 0x4d, 0x4f, 0x44, 0x45, 0x5d, 
-0x20, 0x74, 0x6f, 0x20, 0x63, 0x61, 0x6e, 0x63, 
-0x65, 0x6c, 0x00, 0x45, 0x72, 0x72, 0x6f, 0x72, 
-0x73, 0x3a, 0x00, 0x30, 0x4b, 0x20, 0x6f, 0x66, 
-0x20, 0x32, 0x35, 0x36, 0x4b, 0x00, 0x00, 0x04, 
-0x00, 0xd4, 0x3f, 0x30, 0x30, 0x30, 0x30, 0x3f, 
-0xd4, 0x93, 0x1b, 
 };
-static unsigned int romDumpSize83 = 771;
+static unsigned int romDumpSize83 = 0;
--- a/src/rom83p.h
+++ b/src/rom83p.h
@@ -1,114 +1,3 @@
 static unsigned char romDump8Xp[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x38, 0x33, 0x46, 0x2a, 
-0x1a, 0x0a, 0x00, 0x43, 0x72, 0x65, 0x61, 0x74, 
-0x65, 0x64, 0x20, 0x62, 0x79, 0x20, 0x74, 0x69, 
-0x70, 0x61, 0x63, 0x6b, 0x20, 0x30, 0x2e, 0x31, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x3e, 0x03, 0x0d, 
-0x00, 0x2d, 0x03, 0x06, 0x52, 0x4f, 0x4d, 0x44, 
-0x55, 0x4d, 0x50, 0x00, 0x00, 0x00, 0x2d, 0x03, 
-0x2b, 0x03, 0xbb, 0x6d, 0xdb, 0x02, 0x07, 0xe6, 
-0x01, 0xd3, 0x20, 0xfd, 0xcb, 0x1f, 0xae, 0xfd, 
-0xcb, 0x08, 0x9e, 0xdb, 0x06, 0x32, 0xec, 0x86, 
-0xcd, 0xf1, 0x9e, 0x3a, 0xec, 0x86, 0xd3, 0x06, 
-0xdb, 0x02, 0x87, 0x30, 0x10, 0x3e, 0x01, 0xd3, 
-0x20, 0xdb, 0x21, 0xe6, 0x03, 0x3e, 0x10, 0x28, 
-0x01, 0x87, 0x32, 0xbc, 0xa0, 0x21, 0x00, 0x00, 
-0x22, 0xef, 0x86, 0x22, 0xf1, 0x86, 0xef, 0x70, 
-0x45, 0xef, 0x46, 0x45, 0x21, 0x00, 0x00, 0x22, 
-0x4b, 0x84, 0x21, 0x8c, 0xa0, 0xef, 0x0a, 0x45, 
-0x11, 0x04, 0x00, 0xed, 0x53, 0x4b, 0x84, 0xef, 
-0x0a, 0x45, 0xfb, 0x76, 0xef, 0x18, 0x40, 0xfe, 
-0x37, 0x28, 0x2f, 0xcd, 0x5f, 0xa0, 0x28, 0xf2, 
-0xcd, 0xb6, 0x9f, 0x38, 0x62, 0xed, 0x5b, 0xff, 
-0x86, 0x21, 0x55, 0xaa, 0xb7, 0xed, 0x52, 0x28, 
-0x09, 0x7a, 0xb7, 0x20, 0x52, 0x43, 0x10, 0x0a, 
-0x18, 0xd8, 0x21, 0x01, 0x00, 0xcd, 0xec, 0x9f, 
-0x18, 0xd0, 0x10, 0x10, 0x21, 0x02, 0x00, 0xcd, 
-0xec, 0x9f, 0xef, 0x46, 0x45, 0x21, 0x00, 0x00, 
-0x22, 0x4b, 0x84, 0xc9, 0x10, 0x2f, 0x21, 0xba, 
-0xa0, 0x22, 0xf7, 0x86, 0x21, 0x04, 0x00, 0x22, 
-0xf5, 0x86, 0x21, 0x03, 0x00, 0xcd, 0xf4, 0x9f, 
-0x21, 0x02, 0x04, 0x22, 0x4b, 0x84, 0x21, 0xb4, 
-0xa0, 0xef, 0x0a, 0x45, 0x2a, 0xbb, 0xa0, 0xcb, 
-0x3c, 0xcb, 0x3c, 0xef, 0x07, 0x45, 0x3e, 0x4b, 
-0xef, 0x04, 0x45, 0x18, 0x8d, 0x10, 0x1b, 0x21, 
-0x07, 0x00, 0x22, 0x4b, 0x84, 0x21, 0xac, 0xa0, 
-0xef, 0x0a, 0x45, 0x2a, 0xef, 0x86, 0x23, 0x22, 
-0xef, 0x86, 0xef, 0x07, 0x45, 0xcd, 0x48, 0xa0, 
-0x18, 0xe1, 0x10, 0xe3, 0x2a, 0x03, 0x87, 0x3a, 
-0x05, 0x87, 0xcb, 0x24, 0x17, 0xcb, 0x24, 0x17, 
-0xd3, 0x06, 0x37, 0xcb, 0x1c, 0xcb, 0x3c, 0x22, 
-0xf7, 0x86, 0xcd, 0xac, 0x9f, 0x20, 0x19, 0x67, 
-0x6f, 0x22, 0xfb, 0x86, 0x21, 0x00, 0x04, 0x22, 
-0xf9, 0x86, 0x21, 0xf9, 0x86, 0x22, 0xf7, 0x86, 
-0x21, 0x04, 0x00, 0x11, 0x07, 0x00, 0x18, 0x06, 
-0x21, 0x00, 0x04, 0x11, 0x06, 0x00, 0x22, 0xf5, 
-0x86, 0xeb, 0xcd, 0xf4, 0x9f, 0x3a, 0xec, 0x86, 
-0xd3, 0x06, 0x38, 0x9b, 0x21, 0x02, 0x00, 0x22, 
-0x4b, 0x84, 0x2a, 0xf1, 0x86, 0x23, 0x22, 0xf1, 
-0x86, 0xef, 0x07, 0x45, 0x3e, 0x4b, 0xef, 0x04, 
-0x45, 0x18, 0x9d, 0xe9, 0x01, 0x13, 0x01, 0x01, 
-0x01, 0x13, 0x01, 0x00, 0x02, 0x37, 0x01, 0x03, 
-0x02, 0x37, 0x01, 0x03, 0x02, 0x38, 0x01, 0x03, 
-0xef, 0x6f, 0x4c, 0x4f, 0xef, 0xba, 0x80, 0x6f, 
-0xc6, 0xfc, 0xd8, 0x3c, 0x20, 0x05, 0xcb, 0x40, 
-0x20, 0x01, 0x2c, 0x26, 0x00, 0x29, 0x29, 0x11, 
-0xdd, 0x9e, 0x19, 0x79, 0xcd, 0xa5, 0x9f, 0xd8, 
-0x23, 0xef, 0xb7, 0x80, 0xcd, 0xa5, 0x9f, 0xd8, 
-0xf3, 0xfd, 0xe5, 0x11, 0x09, 0x7b, 0x7b, 0x32, 
-0xee, 0x83, 0xd3, 0x06, 0x1a, 0x47, 0xc5, 0xd5, 
-0xcd, 0x91, 0x9f, 0x7b, 0xef, 0xc6, 0x80, 0x21, 
-0x80, 0x9f, 0x11, 0x38, 0x98, 0xef, 0xda, 0x47, 
-0xed, 0x73, 0x3c, 0x98, 0xcd, 0x44, 0x9f, 0xf1, 
-0xc1, 0x18, 0x4a, 0x3a, 0x94, 0x50, 0x57, 0x21, 
-0x00, 0x81, 0x01, 0x30, 0x01, 0x3e, 0xed, 0xed, 
-0xb1, 0xc0, 0x7e, 0xd6, 0xa0, 0x20, 0xf6, 0x4e, 
-0x47, 0x23, 0x22, 0xdb, 0x84, 0x2a, 0x92, 0x50, 
-0x3c, 0x32, 0x34, 0x98, 0xb7, 0xc4, 0x91, 0x9f, 
-0x3e, 0x3a, 0xed, 0xb1, 0xc0, 0x11, 0x63, 0x9f, 
-0x1a, 0xbe, 0x20, 0xf4, 0x23, 0x13, 0xfe, 0xc4, 
-0x20, 0xf6, 0xef, 0x09, 0x40, 0xfd, 0x21, 0x31, 
-0x00, 0xe9, 0x31, 0x00, 0x00, 0xe1, 0xf1, 0x77, 
-0xbe, 0x20, 0xfd, 0x36, 0xf0, 0xfd, 0xe1, 0xc9, 
-0xdb, 0x02, 0x87, 0x3e, 0x1f, 0x30, 0x09, 0xdb, 
-0x21, 0xe6, 0x03, 0x3e, 0x7f, 0x20, 0x01, 0x1f, 
-0xa2, 0xd3, 0x06, 0xc9, 0xbe, 0x23, 0x3f, 0xc0, 
-0x7e, 0xb8, 0xc9, 0x7e, 0x01, 0x00, 0x04, 0xed, 
-0xa1, 0xe0, 0x28, 0xfb, 0xc9, 0x21, 0x00, 0x00, 
-0x11, 0xff, 0x86, 0x06, 0x04, 0xcd, 0x31, 0xa0, 
-0xd8, 0x3a, 0x02, 0x87, 0xc6, 0xff, 0xd8, 0x3a, 
-0x01, 0x87, 0xb7, 0x28, 0x0b, 0x47, 0xc6, 0xbf, 
-0xd8, 0x11, 0x03, 0x87, 0xcd, 0x31, 0xa0, 0xd8, 
-0xe5, 0x11, 0xfd, 0x86, 0x06, 0x02, 0xcd, 0x31, 
-0xa0, 0xd1, 0xd8, 0x2a, 0xfd, 0x86, 0xed, 0x52, 
-0xc8, 0x37, 0xc9, 0xe5, 0x21, 0x00, 0x00, 0x22, 
-0xf5, 0x86, 0xe1, 0x22, 0xf3, 0x86, 0x21, 0x00, 
-0x00, 0x11, 0xf3, 0x86, 0x01, 0x04, 0x00, 0xcd, 
-0x1b, 0xa0, 0xd8, 0xed, 0x5b, 0xf7, 0x86, 0xed, 
-0x4b, 0xf5, 0x86, 0xcd, 0x1b, 0xa0, 0xd8, 0x22, 
-0xfd, 0x86, 0x11, 0xfd, 0x86, 0x01, 0x02, 0x00, 
-0x18, 0x00, 0x78, 0xb1, 0xc8, 0x0b, 0xc5, 0x1a, 
-0x13, 0x4f, 0x06, 0x00, 0x09, 0xd5, 0xe5, 0xcd, 
-0x7b, 0xa0, 0xe1, 0xd1, 0xc1, 0xd8, 0x18, 0xea, 
-0xc5, 0xe5, 0xd5, 0xcd, 0x6f, 0xa0, 0xd1, 0xe1, 
-0x38, 0x0b, 0x06, 0x00, 0x09, 0x79, 0x12, 0x13, 
-0xc1, 0x10, 0xed, 0xb7, 0xc5, 0xc1, 0xc9, 0x06, 
-0xff, 0xf5, 0xdb, 0x02, 0xe6, 0x80, 0xd3, 0x08, 
-0x3e, 0x00, 0xd3, 0x00, 0xfb, 0x76, 0x76, 0x10, 
-0xfb, 0xaf, 0xd3, 0x08, 0xf1, 0xc9, 0xdb, 0x00, 
-0x2f, 0xe6, 0x03, 0xc0, 0xdb, 0x02, 0xe6, 0x80, 
-0xc8, 0xdb, 0x09, 0xe6, 0x18, 0xc9, 0x21, 0x8a, 
-0xa0, 0xcd, 0x59, 0x00, 0xef, 0x03, 0x4f, 0x57, 
-0x18, 0x09, 0x21, 0x8a, 0xa0, 0xcd, 0x59, 0x00, 
-0xef, 0xe5, 0x4e, 0xcd, 0x5c, 0x00, 0x4a, 0xb7, 
-0xc9, 0x37, 0xc9, 0x44, 0x75, 0x6d, 0x70, 0x69, 
-0x6e, 0x67, 0x20, 0x52, 0x4f, 0x4d, 0x2e, 0x2e, 
-0x2e, 0x00, 0xc1, 0x4d, 0x4f, 0x44, 0x45, 0x5d, 
-0x20, 0x74, 0x6f, 0x20, 0x63, 0x61, 0x6e, 0x63, 
-0x65, 0x6c, 0x00, 0x45, 0x72, 0x72, 0x6f, 0x72, 
-0x73, 0x3a, 0x00, 0x30, 0x4b, 0x20, 0x6f, 0x66, 
-0x20, 0x00, 0x00, 0x08, 0x00, 0x66, 0x5b, 
 };
-static unsigned int romDumpSize8Xp = 887;
+static unsigned int romDumpSize8Xp = 0;
--- a/src/rom84p.h
+++ b/src/rom84p.h
@@ -1,196 +1,3 @@
 static unsigned char romDump84p[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x38, 0x33, 0x46, 0x2a, 
-0x1a, 0x0a, 0x00, 0x43, 0x72, 0x65, 0x61, 0x74, 
-0x65, 0x64, 0x20, 0x62, 0x79, 0x20, 0x74, 0x69, 
-0x70, 0x61, 0x63, 0x6b, 0x20, 0x30, 0x2e, 0x31, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0xcb, 0x05, 0x0d, 
-0x00, 0xba, 0x05, 0x06, 0x52, 0x4f, 0x4d, 0x44, 
-0x55, 0x4d, 0x50, 0x00, 0x00, 0x00, 0xba, 0x05, 
-0xb8, 0x05, 0xbb, 0x6d, 0xef, 0x70, 0x45, 0xdb, 
-0x06, 0xf5, 0xcd, 0x90, 0xa2, 0xf1, 0xd3, 0x06, 
-0xed, 0x73, 0x72, 0x98, 0xcd, 0xc2, 0x9d, 0xcd, 
-0xe7, 0xa0, 0x30, 0x20, 0xcd, 0x57, 0xa0, 0x30, 
-0x1b, 0xed, 0x7b, 0x72, 0x98, 0xcd, 0xc2, 0x9d, 
-0x21, 0x2c, 0x9f, 0xef, 0x0a, 0x45, 0xef, 0x72, 
-0x49, 0xef, 0x40, 0x45, 0xef, 0x58, 0x45, 0xfd, 
-0xcb, 0x09, 0xa6, 0xc9, 0x21, 0xd5, 0x9e, 0xef, 
-0x0a, 0x45, 0xfd, 0xcb, 0x41, 0xae, 0x21, 0x00, 
-0x80, 0xcd, 0x0e, 0xa0, 0x38, 0xd3, 0x2a, 0x00, 
-0x80, 0x11, 0x02, 0x00, 0xef, 0x0c, 0x40, 0x28, 
-0x1a, 0x11, 0x03, 0x00, 0xef, 0x0c, 0x40, 0x28, 
-0x1d, 0x11, 0x05, 0x00, 0xef, 0x0c, 0x40, 0x28, 
-0x4c, 0x11, 0x55, 0xaa, 0xef, 0x0c, 0x40, 0x28, 
-0x36, 0x18, 0xd3, 0x11, 0x02, 0x00, 0x01, 0x00, 
-0x00, 0xcd, 0xa2, 0x9f, 0x18, 0xb3, 0x21, 0x74, 
-0x98, 0x36, 0x00, 0x23, 0x36, 0x00, 0x23, 0xdb, 
-0x21, 0xe6, 0x03, 0x3e, 0x10, 0x28, 0x01, 0x87, 
-0x77, 0x23, 0x36, 0x00, 0x3e, 0x81, 0x21, 0x74, 
-0x58, 0x11, 0x03, 0x00, 0x01, 0x04, 0x00, 0xcd, 
-0xa2, 0x9f, 0xda, 0xb2, 0x9d, 0x18, 0x9f, 0x11, 
-0x01, 0x00, 0x01, 0x00, 0x00, 0xcd, 0xa2, 0x9f, 
-0xda, 0xb2, 0x9d, 0x18, 0x91, 0xdd, 0x21, 0x04, 
-0x80, 0xdd, 0x6e, 0x00, 0xdd, 0x66, 0x01, 0xdd, 
-0x5e, 0x02, 0xdd, 0x56, 0x03, 0x01, 0x02, 0x04, 
-0xed, 0x43, 0x4b, 0x84, 0x7b, 0xcd, 0x30, 0xa0, 
-0x7c, 0xcd, 0x30, 0xa0, 0x7d, 0xcd, 0x30, 0xa0, 
-0x7b, 0x87, 0x87, 0x11, 0x00, 0x40, 0xb7, 0xed, 
-0x52, 0x38, 0x03, 0x3c, 0x18, 0xf5, 0x19, 0x11, 
-0x00, 0x40, 0x19, 0xe5, 0x32, 0xeb, 0x83, 0xdb, 
-0x06, 0xf5, 0x3a, 0xeb, 0x83, 0xd3, 0x06, 0x56, 
-0x01, 0x00, 0x04, 0x7e, 0xba, 0x20, 0x06, 0x23, 
-0x0b, 0x78, 0xb1, 0x20, 0xf6, 0xc1, 0x78, 0xd3, 
-0x06, 0xe1, 0x28, 0x12, 0x3a, 0xeb, 0x83, 0x11, 
-0x06, 0x00, 0x01, 0x00, 0x04, 0xcd, 0xa2, 0x9f, 
-0xda, 0xb2, 0x9d, 0xc3, 0xd7, 0x9d, 0xdd, 0x21, 
-0x78, 0x84, 0xdd, 0x36, 0x00, 0x00, 0xdd, 0x36, 
-0x01, 0x04, 0xdd, 0x72, 0x02, 0xdd, 0x72, 0x03, 
-0x3e, 0x81, 0x21, 0x78, 0x44, 0x11, 0x07, 0x00, 
-0x01, 0x04, 0x00, 0xcd, 0xa2, 0x9f, 0xda, 0xb2, 
-0x9d, 0xc3, 0xd7, 0x9d, 0x44, 0x75, 0x6d, 0x70, 
-0x69, 0x6e, 0x67, 0xce, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x50, 0x72, 0x65, 0x73, 
-0x73, 0x20, 0xc1, 0x4f, 0x4e, 0x5d, 0x20, 0x74, 
-0x6f, 0x20, 0x20, 0x20, 0x61, 0x62, 0x6f, 0x72, 
-0x74, 0x2e, 0x00, 0x45, 0x52, 0x52, 0x4f, 0x52, 
-0x21, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x54, 0x68, 0x65, 0x72, 0x65, 
-0x20, 0x77, 0x61, 0x73, 0x20, 0x61, 0x6e, 0x20, 
-0x20, 0x20, 0x20, 0x75, 0x6e, 0x6b, 0x6e, 0x6f, 
-0x77, 0x6e, 0x20, 0x55, 0x53, 0x42, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x65, 0x72, 0x72, 0x6f, 0x72, 
-0x2e, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x20, 0x50, 0x72, 0x65, 0x73, 0x73, 
-0x20, 0x61, 0x6e, 0x79, 0x20, 0x6b, 0x65, 0x79, 
-0x20, 0x74, 0x6f, 0x71, 0x75, 0x69, 0x74, 0x2e, 
-0x00, 0xf3, 0xc5, 0x32, 0xeb, 0x83, 0xdb, 0x06, 
-0xf5, 0x3e, 0x83, 0xd3, 0x06, 0xe5, 0x21, 0x00, 
-0x40, 0x73, 0x23, 0x72, 0x23, 0x71, 0x23, 0x70, 
-0x23, 0xeb, 0xe1, 0x78, 0xb1, 0x28, 0x25, 0xc5, 
-0x3a, 0xeb, 0x83, 0xd3, 0x06, 0x7e, 0xf5, 0x3e, 
-0x83, 0xd3, 0x06, 0xf1, 0x12, 0x23, 0xcb, 0x7c, 
-0x28, 0x0b, 0xcb, 0xbc, 0xcb, 0xf4, 0x3a, 0xeb, 
-0x83, 0x3c, 0x32, 0xeb, 0x83, 0x13, 0x0b, 0x78, 
-0xb1, 0x20, 0xdd, 0xc1, 0x03, 0x03, 0x03, 0x03, 
-0x21, 0x00, 0x40, 0x11, 0x00, 0x00, 0x7e, 0x23, 
-0x83, 0x5f, 0x30, 0x01, 0x14, 0x0b, 0x78, 0xb1, 
-0x20, 0xf4, 0x73, 0x23, 0x72, 0xf1, 0xd3, 0x06, 
-0xd1, 0x13, 0x13, 0x13, 0x13, 0x13, 0x13, 0x21, 
-0x00, 0x80, 0xc3, 0xfa, 0xa0, 0x01, 0x04, 0x00, 
-0xe5, 0xcd, 0x0a, 0xa1, 0xe1, 0xd8, 0x23, 0x23, 
-0x4e, 0x23, 0x46, 0x23, 0x78, 0xb1, 0x28, 0x09, 
-0xe5, 0xc5, 0xcd, 0x0a, 0xa1, 0xc1, 0xe1, 0xd8, 
-0x09, 0x01, 0x02, 0x00, 0xc3, 0x0a, 0xa1, 0xdd, 
-0xe5, 0xf5, 0xe5, 0xc5, 0xf5, 0x0f, 0x0f, 0x0f, 
-0x0f, 0xcd, 0x47, 0xa0, 0xf1, 0xcd, 0x47, 0xa0, 
-0xc1, 0xe1, 0xf1, 0xdd, 0xe1, 0xc9, 0xe6, 0x0f, 
-0xfe, 0x0a, 0x30, 0x04, 0xc6, 0x30, 0x18, 0x02, 
-0xc6, 0x37, 0xef, 0x04, 0x45, 0xc9, 0xef, 0x0e, 
-0x81, 0xd8, 0x3e, 0x80, 0xd3, 0x57, 0xaf, 0xd3, 
-0x4c, 0x3e, 0x01, 0xd3, 0x5b, 0xaf, 0xdb, 0x4c, 
-0x3e, 0x02, 0xd3, 0x54, 0x3e, 0x20, 0xd3, 0x4a, 
-0xaf, 0xd3, 0x4b, 0xdb, 0x3a, 0xcb, 0x5f, 0x28, 
-0x04, 0x3e, 0x20, 0xd3, 0x4b, 0xaf, 0xd3, 0x54, 
-0x3e, 0x42, 0xd3, 0x36, 0xaf, 0xd3, 0x37, 0x3e, 
-0x01, 0xd3, 0x38, 0xdb, 0x04, 0xcb, 0x7f, 0x28, 
-0xfa, 0xdb, 0x3a, 0xcb, 0x5f, 0x28, 0x04, 0x3e, 
-0x44, 0xd3, 0x54, 0x3e, 0xc4, 0xd3, 0x54, 0x3e, 
-0x08, 0xd3, 0x4c, 0x11, 0xff, 0xff, 0x1b, 0x7a, 
-0xb3, 0x37, 0xc8, 0xdb, 0x4c, 0xfe, 0x1a, 0x28, 
-0x04, 0xfe, 0x5a, 0x20, 0xf1, 0x3e, 0xff, 0xd3, 
-0x87, 0xaf, 0xd3, 0x92, 0xdb, 0x87, 0x3e, 0x0e, 
-0xd3, 0x89, 0x3e, 0x05, 0xd3, 0x8b, 0xdb, 0x81, 
-0xf6, 0x00, 0xd3, 0x81, 0xdb, 0x54, 0xf6, 0x01, 
-0xd3, 0x54, 0x06, 0x05, 0x11, 0xff, 0xff, 0x1b, 
-0x7a, 0xb3, 0x28, 0x06, 0xdb, 0x8c, 0xb7, 0x28, 
-0xf6, 0xc9, 0x10, 0xf0, 0x37, 0xc9, 0xdb, 0x02, 
-0xcb, 0x6f, 0x37, 0xc8, 0x3a, 0x4d, 0x00, 0xb7, 
-0xcb, 0x6f, 0xc8, 0xcb, 0x77, 0x37, 0xc0, 0x3f, 
-0xc9, 0xfd, 0xcb, 0x40, 0xd6, 0xfd, 0xcb, 0x43, 
-0x86, 0xef, 0xf2, 0x50, 0xfd, 0xcb, 0x41, 0x86, 
-0xc9, 0xfb, 0x76, 0xdb, 0x04, 0xcb, 0x5f, 0x37, 
-0xc8, 0xfd, 0xcb, 0x41, 0x6e, 0x28, 0xf2, 0xc5, 
-0xf5, 0xaf, 0x32, 0x34, 0x98, 0x18, 0x28, 0xdb, 
-0x04, 0xcb, 0x5f, 0x28, 0x47, 0xfd, 0xcb, 0x41, 
-0x6e, 0x20, 0x1c, 0xdb, 0x84, 0xcb, 0x57, 0x20, 
-0x04, 0xcb, 0x4f, 0x28, 0x0a, 0xfd, 0xcb, 0x41, 
-0xee, 0xaf, 0x32, 0x27, 0x9c, 0x18, 0x08, 0xdb, 
-0x86, 0xcb, 0x6f, 0x20, 0x27, 0x18, 0xd8, 0xc5, 
-0xe5, 0x21, 0x40, 0x00, 0xb7, 0xed, 0x42, 0xe1, 
-0x41, 0x30, 0x02, 0x06, 0x40, 0xcd, 0x71, 0xa1, 
-0xeb, 0xe1, 0x38, 0x10, 0xb7, 0x06, 0x00, 0xed, 
-0x42, 0x28, 0x05, 0x44, 0x4d, 0xeb, 0x18, 0xb7, 
-0xc1, 0x78, 0xc1, 0xc9, 0xf1, 0xc1, 0x37, 0xc9, 
-0x78, 0xb7, 0xc8, 0x3e, 0x40, 0xb8, 0xd8, 0x78, 
-0x32, 0x80, 0x9c, 0x3a, 0x27, 0x9c, 0xb7, 0x28, 
-0x07, 0xb8, 0x30, 0x04, 0x47, 0x32, 0x80, 0x9c, 
-0xdb, 0x8f, 0xcb, 0x57, 0xca, 0x27, 0xa2, 0xaf, 
-0xd3, 0x5b, 0x3a, 0x27, 0x9c, 0xb7, 0x20, 0x3d, 
-0x3e, 0x01, 0xd3, 0x8e, 0x3e, 0x21, 0xd3, 0x9a, 
-0xaf, 0xdb, 0x9a, 0x3e, 0x08, 0xd3, 0x93, 0xaf, 
-0xdb, 0x93, 0x3e, 0xff, 0xd3, 0x89, 0xaf, 0xd3, 
-0x95, 0xdb, 0x89, 0x3e, 0xa1, 0xd3, 0x8b, 0xdb, 
-0x94, 0xcb, 0x57, 0xc2, 0x6d, 0xa2, 0xcb, 0x77, 
-0xc2, 0x6d, 0xa2, 0x3e, 0x01, 0xd3, 0x5b, 0x3a, 
-0x27, 0x9c, 0xb7, 0x20, 0x08, 0x3e, 0x01, 0xd3, 
-0x8e, 0xdb, 0x94, 0xdb, 0x96, 0xf5, 0x3a, 0x80, 
-0x9c, 0x47, 0x0e, 0x00, 0xdb, 0xa1, 0x77, 0x23, 
-0x0c, 0x10, 0xf9, 0x3e, 0x01, 0xd3, 0x8e, 0xdb, 
-0x94, 0xcb, 0x57, 0x20, 0x7e, 0xcb, 0x77, 0x20, 
-0x7a, 0xf1, 0x91, 0x32, 0x27, 0x9c, 0xfd, 0xcb, 
-0x41, 0xee, 0xc0, 0x3e, 0x01, 0xd3, 0x8e, 0x3e, 
-0x21, 0xd3, 0x9a, 0x3e, 0x08, 0xd3, 0x93, 0xaf, 
-0xdb, 0x93, 0x3e, 0x0e, 0xd3, 0x89, 0xaf, 0xd3, 
-0x95, 0xfd, 0xcb, 0x41, 0xae, 0xaf, 0x32, 0x27, 
-0x9c, 0xfd, 0xcb, 0x41, 0x86, 0x3e, 0x20, 0xd3, 
-0x94, 0x3e, 0x01, 0xd3, 0x5b, 0xc9, 0x3e, 0x02, 
-0xd3, 0x8e, 0xdb, 0x94, 0xcb, 0x77, 0x28, 0x0a, 
-0xe6, 0xdf, 0xd3, 0x94, 0xf1, 0xef, 0x57, 0x52, 
-0x18, 0x32, 0x3a, 0x27, 0x9c, 0xb7, 0x20, 0x02, 
-0xdb, 0x96, 0xf5, 0x0e, 0x00, 0xdb, 0xa2, 0x77, 
-0x23, 0x0c, 0x10, 0xf9, 0x3e, 0x02, 0xd3, 0x8e, 
-0xf1, 0x91, 0x32, 0x27, 0x9c, 0xc0, 0xaf, 0x32, 
-0x27, 0x9c, 0xdb, 0x94, 0xe6, 0xfe, 0xd3, 0x94, 
-0xfd, 0xcb, 0x41, 0xae, 0x3e, 0xa1, 0x32, 0x8b, 
-0x00, 0x18, 0x06, 0xf1, 0xfd, 0xcb, 0x41, 0xae, 
-0x37, 0xfb, 0x3e, 0x01, 0xd3, 0x5b, 0xfd, 0xcb, 
-0x41, 0x86, 0xc9, 0x01, 0x13, 0x01, 0x01, 0x01, 
-0x13, 0x01, 0x00, 0x02, 0x37, 0x01, 0x03, 0x02, 
-0x37, 0x01, 0x03, 0x02, 0x38, 0x01, 0x03, 0xef, 
-0x6f, 0x4c, 0x4f, 0xef, 0xba, 0x80, 0x6f, 0xc6, 
-0xfc, 0xd8, 0x3c, 0x20, 0x05, 0xcb, 0x40, 0x20, 
-0x01, 0x2c, 0x26, 0x00, 0x29, 0x29, 0x11, 0x7c, 
-0xa2, 0x19, 0x79, 0xcd, 0x44, 0xa3, 0xd8, 0x23, 
-0xef, 0xb7, 0x80, 0xcd, 0x44, 0xa3, 0xd8, 0xf3, 
-0xfd, 0xe5, 0x11, 0x09, 0x7b, 0x7b, 0x32, 0xee, 
-0x83, 0xd3, 0x06, 0x1a, 0x47, 0xc5, 0xd5, 0xcd, 
-0x30, 0xa3, 0x7b, 0xef, 0xc6, 0x80, 0x21, 0x1f, 
-0xa3, 0x11, 0x38, 0x98, 0xef, 0xda, 0x47, 0xed, 
-0x73, 0x3c, 0x98, 0xcd, 0xe3, 0xa2, 0xf1, 0xc1, 
-0x18, 0x4a, 0x3a, 0x94, 0x50, 0x57, 0x21, 0x00, 
-0x81, 0x01, 0x30, 0x01, 0x3e, 0xed, 0xed, 0xb1, 
-0xc0, 0x7e, 0xd6, 0xa0, 0x20, 0xf6, 0x4e, 0x47, 
-0x23, 0x22, 0xdb, 0x84, 0x2a, 0x92, 0x50, 0x3c, 
-0x32, 0x34, 0x98, 0xb7, 0xc4, 0x30, 0xa3, 0x3e, 
-0x3a, 0xed, 0xb1, 0xc0, 0x11, 0x02, 0xa3, 0x1a, 
-0xbe, 0x20, 0xf4, 0x23, 0x13, 0xfe, 0xc4, 0x20, 
-0xf6, 0xef, 0x09, 0x40, 0xfd, 0x21, 0x31, 0x00, 
-0xe9, 0x31, 0x00, 0x00, 0xe1, 0xf1, 0x77, 0xbe, 
-0x20, 0xfd, 0x36, 0xf0, 0xfd, 0xe1, 0xc9, 0xdb, 
-0x02, 0x87, 0x3e, 0x1f, 0x30, 0x09, 0xdb, 0x21, 
-0xe6, 0x03, 0x3e, 0x7f, 0x20, 0x01, 0x1f, 0xa2, 
-0xd3, 0x06, 0xc9, 0xbe, 0x23, 0x3f, 0xc0, 0x7e, 
-0xb8, 0xc9, 0x6f, 0x6d, 
 };
-static unsigned int romDumpSize84p = 1540;
+static unsigned int romDumpSize84p = 0;
--- a/src/rom85.h
+++ b/src/rom85.h
@@ -1,156 +1,3 @@
 static unsigned char romDump85[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x38, 0x35, 0x2a, 0x2a, 
-0x1a, 0x0c, 0x00, 0x43, 0x72, 0x65, 0x61, 0x74, 
-0x65, 0x64, 0x20, 0x62, 0x79, 0x20, 0x74, 0x69, 
-0x70, 0x61, 0x63, 0x6b, 0x20, 0x30, 0x2e, 0x31, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x89, 0x04, 0x07, 
-0x00, 0x7e, 0x04, 0x0c, 0x03, 0x24, 0x52, 0x44, 
-0x7e, 0x04, 0x7c, 0x04, 0x11, 0x43, 0x01, 0x19, 
-0x11, 0xc1, 0x86, 0x01, 0x01, 0x02, 0xed, 0xb0, 
-0x22, 0x1a, 0x88, 0x11, 0xea, 0xfc, 0x19, 0xcd, 
-0xc1, 0x86, 0x2a, 0xe3, 0x8b, 0x22, 0xe5, 0x8b, 
-0x2a, 0xdf, 0x8b, 0x22, 0xe1, 0x8b, 0xe1, 0xe1, 
-0xe1, 0xf1, 0xd3, 0x05, 0x3e, 0x06, 0x32, 0x37, 
-0x83, 0xc9, 0xfd, 0xcb, 0x08, 0x9e, 0xdb, 0x05, 
-0x32, 0x41, 0x86, 0x21, 0x00, 0x00, 0x22, 0x44, 
-0x86, 0x22, 0x46, 0x86, 0xfd, 0xcb, 0x12, 0x86, 
-0xcd, 0x09, 0x88, 0x21, 0x00, 0x00, 0x22, 0x0c, 
-0x80, 0x21, 0x8c, 0x88, 0xcd, 0x00, 0x88, 0x11, 
-0x04, 0x00, 0xed, 0x53, 0x0c, 0x80, 0xcd, 0x00, 
-0x88, 0xfb, 0x76, 0xf3, 0x21, 0x00, 0x80, 0x7e, 
-0x36, 0x00, 0xfe, 0x37, 0x28, 0x2f, 0xcd, 0x6c, 
-0x87, 0x28, 0xee, 0xcd, 0xcc, 0x86, 0x38, 0x53, 
-0xed, 0x5b, 0x54, 0x86, 0x21, 0x55, 0xaa, 0xb7, 
-0xed, 0x52, 0x28, 0x09, 0x7a, 0xb7, 0x20, 0x43, 
-0x43, 0x10, 0x0a, 0x18, 0xd4, 0x21, 0x01, 0x00, 
-0xcd, 0x02, 0x87, 0x18, 0xcc, 0x10, 0x10, 0x21, 
-0x02, 0x00, 0xcd, 0x02, 0x87, 0xcd, 0x09, 0x88, 
-0x21, 0x00, 0x00, 0x22, 0x0c, 0x80, 0xc9, 0x10, 
-0x20, 0x21, 0xbe, 0x88, 0x22, 0x4c, 0x86, 0x21, 
-0x04, 0x00, 0x22, 0x4a, 0x86, 0x21, 0x03, 0x00, 
-0xcd, 0x0a, 0x87, 0x21, 0x02, 0x04, 0x22, 0x0c, 
-0x80, 0x21, 0xb4, 0x88, 0xcd, 0x00, 0x88, 0x18, 
-0x98, 0x10, 0x1b, 0x21, 0x07, 0x00, 0x22, 0x0c, 
-0x80, 0x21, 0xac, 0x88, 0xcd, 0x00, 0x88, 0x2a, 
-0x44, 0x86, 0x23, 0x22, 0x44, 0x86, 0xcd, 0xd4, 
-0x87, 0xcd, 0x5e, 0x87, 0x18, 0xe1, 0x10, 0xe3, 
-0x2a, 0x58, 0x86, 0x3a, 0x5a, 0x86, 0xcb, 0x24, 
-0x17, 0xcb, 0x24, 0x17, 0xd3, 0x05, 0x37, 0xcb, 
-0x1c, 0xcb, 0x3c, 0x22, 0x4c, 0x86, 0xcd, 0xc2, 
-0x86, 0x20, 0x19, 0x67, 0x6f, 0x22, 0x50, 0x86, 
-0x21, 0x00, 0x04, 0x22, 0x4e, 0x86, 0x21, 0x4e, 
-0x86, 0x22, 0x4c, 0x86, 0x21, 0x04, 0x00, 0x11, 
-0x07, 0x00, 0x18, 0x06, 0x21, 0x00, 0x04, 0x11, 
-0x06, 0x00, 0x22, 0x4a, 0x86, 0xeb, 0xcd, 0x0a, 
-0x87, 0x3a, 0x41, 0x86, 0xd3, 0x05, 0x38, 0x9b, 
-0x21, 0x02, 0x00, 0x22, 0x0c, 0x80, 0x2a, 0x46, 
-0x86, 0x23, 0x22, 0x46, 0x86, 0xcd, 0xd4, 0x87, 
-0x3e, 0x4b, 0xcd, 0x16, 0x88, 0x18, 0x9d, 0xe9, 
-0x7e, 0x01, 0x00, 0x04, 0xed, 0xa1, 0xe0, 0x28, 
-0xfb, 0xc9, 0x21, 0x00, 0x00, 0x11, 0x54, 0x86, 
-0x06, 0x04, 0xcd, 0x47, 0x87, 0xd8, 0x3a, 0x57, 
-0x86, 0xc6, 0xff, 0xd8, 0x3a, 0x56, 0x86, 0xb7, 
-0x28, 0x0b, 0x47, 0xc6, 0xbf, 0xd8, 0x11, 0x58, 
-0x86, 0xcd, 0x47, 0x87, 0xd8, 0xe5, 0x11, 0x52, 
-0x86, 0x06, 0x02, 0xcd, 0x47, 0x87, 0xd1, 0xd8, 
-0x2a, 0x52, 0x86, 0xed, 0x52, 0xc8, 0x37, 0xc9, 
-0xe5, 0x21, 0x00, 0x00, 0x22, 0x4a, 0x86, 0xe1, 
-0x22, 0x48, 0x86, 0x21, 0x00, 0x00, 0x11, 0x48, 
-0x86, 0x01, 0x04, 0x00, 0xcd, 0x31, 0x87, 0xd8, 
-0xed, 0x5b, 0x4c, 0x86, 0xed, 0x4b, 0x4a, 0x86, 
-0xcd, 0x31, 0x87, 0xd8, 0x22, 0x52, 0x86, 0x11, 
-0x52, 0x86, 0x01, 0x02, 0x00, 0x18, 0x00, 0x78, 
-0xb1, 0xc8, 0x0b, 0xc5, 0x1a, 0x13, 0x4f, 0x06, 
-0x00, 0x09, 0xd5, 0xe5, 0xcd, 0xa8, 0x87, 0xe1, 
-0xd1, 0xc1, 0xd8, 0x18, 0xea, 0xc5, 0xe5, 0xd5, 
-0xcd, 0x72, 0x87, 0xd1, 0xe1, 0x38, 0x0b, 0x06, 
-0x00, 0x09, 0x79, 0x12, 0x13, 0xc1, 0x10, 0xed, 
-0xb7, 0xc5, 0xc1, 0xc9, 0x06, 0xff, 0xf5, 0x3e, 
-0xc0, 0xd3, 0x07, 0xfb, 0x76, 0x76, 0x10, 0xfb, 
-0xf1, 0xc9, 0xdb, 0x07, 0x2f, 0xe6, 0x03, 0xc9, 
-0xf3, 0x3e, 0xc0, 0xd3, 0x07, 0x06, 0x08, 0x16, 
-0xff, 0xcd, 0x98, 0x87, 0xd8, 0xc6, 0xfe, 0x3e, 
-0xe8, 0x30, 0x03, 0xc0, 0x3e, 0xd4, 0xd3, 0x07, 
-0xcb, 0x19, 0x14, 0xcd, 0x98, 0x87, 0xd8, 0x3e, 
-0xc0, 0xd3, 0x07, 0x10, 0xe2, 0xc9, 0x21, 0xff, 
-0xff, 0x2b, 0xdb, 0x07, 0xaa, 0xe6, 0x03, 0xc0, 
-0x7c, 0xb5, 0x20, 0xf5, 0x37, 0xc9, 0xf3, 0x06, 
-0x08, 0xcd, 0xbe, 0x87, 0xd8, 0xcb, 0x19, 0x3e, 
-0xd4, 0x30, 0x02, 0x3e, 0xe8, 0x14, 0xcd, 0xc2, 
-0x87, 0xd8, 0x10, 0xed, 0x3e, 0xc0, 0x16, 0xff, 
-0xd3, 0x07, 0x21, 0xff, 0xff, 0x2b, 0xdb, 0x07, 
-0xaa, 0xe6, 0x03, 0xc8, 0x7c, 0xb5, 0x20, 0xf5, 
-0x37, 0xc9, 0x06, 0x05, 0xeb, 0x3e, 0x20, 0xcd, 
-0x16, 0x88, 0x10, 0xf9, 0x21, 0x0d, 0x80, 0x7e, 
-0xf5, 0x35, 0xeb, 0x01, 0x0a, 0x10, 0xaf, 0x29, 
-0x8f, 0xb9, 0x38, 0x02, 0x91, 0x2c, 0x10, 0xf7, 
-0xeb, 0xc6, 0x30, 0xcd, 0x16, 0x88, 0x35, 0x7a, 
-0xb3, 0x20, 0xe6, 0xf1, 0x77, 0xc9, 0x7e, 0x23, 
-0xb7, 0xc8, 0xcd, 0x16, 0x88, 0x18, 0xf7, 0x21, 
-0x00, 0xfc, 0x75, 0x11, 0x01, 0xfc, 0x01, 0xff, 
-0x03, 0xed, 0xb0, 0xc9, 0xe5, 0xd5, 0xc5, 0x21, 
-0x00, 0x00, 0x01, 0x08, 0x00, 0xbe, 0x28, 0x03, 
-0x09, 0x30, 0xfa, 0x23, 0x01, 0x0c, 0x80, 0x0a, 
-0x0f, 0x5f, 0xf6, 0xfc, 0x57, 0x03, 0x0a, 0x3c, 
-0x02, 0x3d, 0x47, 0x87, 0x80, 0x0f, 0x0f, 0xab, 
-0xe6, 0x1f, 0xab, 0x5f, 0x06, 0x07, 0x3a, 0x0d, 
-0x80, 0xe6, 0x03, 0x28, 0x19, 0x3d, 0x28, 0x1c, 
-0x3d, 0x7e, 0x0f, 0x0f, 0x28, 0x20, 0x4f, 0x1c, 
-0x1a, 0xa9, 0xe6, 0x3f, 0xa9, 0x12, 0x1d, 0x1a, 
-0xa9, 0xe6, 0xf0, 0xa9, 0x18, 0x20, 0x1a, 0xe6, 
-0xc0, 0xb6, 0x18, 0x1a, 0x7e, 0x07, 0x07, 0x4f, 
-0x1a, 0xe6, 0x03, 0xb1, 0x18, 0x10, 0x0f, 0x0f, 
-0x4f, 0x1c, 0x1a, 0xa9, 0xe6, 0x0f, 0xa9, 0x12, 
-0x1d, 0x1a, 0xa9, 0xe6, 0xfc, 0xa9, 0x12, 0x23, 
-0x7b, 0xc6, 0x10, 0x5f, 0x10, 0xb8, 0xc1, 0xd1, 
-0xe1, 0xc9, 0x44, 0x75, 0x6d, 0x70, 0x69, 0x6e, 
-0x67, 0x20, 0x52, 0x4f, 0x4d, 0x2e, 0x2e, 0x2e, 
-0x00, 0x5b, 0x45, 0x58, 0x49, 0x54, 0x5d, 0x20, 
-0x74, 0x6f, 0x20, 0x63, 0x61, 0x6e, 0x63, 0x65, 
-0x6c, 0x00, 0x45, 0x72, 0x72, 0x6f, 0x72, 0x73, 
-0x3a, 0x00, 0x30, 0x4b, 0x20, 0x6f, 0x66, 0x20, 
-0x31, 0x32, 0x38, 0x4b, 0x00, 0x00, 0x02, 0x00, 
-0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x2e, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0c, 0x0c, 
-0x3a, 0x00, 0x0c, 0x0c, 0x00, 0x0c, 0x0c, 0x00, 
-0x5b, 0x0c, 0x08, 0x08, 0x08, 0x08, 0x08, 0x0c, 
-0x5d, 0x18, 0x08, 0x08, 0x08, 0x08, 0x08, 0x18, 
-0x30, 0x1c, 0x22, 0x26, 0x2a, 0x32, 0x22, 0x1c, 
-0x31, 0x08, 0x18, 0x08, 0x08, 0x08, 0x08, 0x1c, 
-0x32, 0x1c, 0x22, 0x02, 0x04, 0x08, 0x10, 0x3e, 
-0x33, 0x3e, 0x04, 0x08, 0x04, 0x02, 0x22, 0x1c, 
-0x34, 0x04, 0x0c, 0x14, 0x24, 0x3e, 0x04, 0x04, 
-0x35, 0x3e, 0x20, 0x3c, 0x02, 0x02, 0x22, 0x1c, 
-0x36, 0x0c, 0x10, 0x20, 0x3c, 0x22, 0x22, 0x1c, 
-0x37, 0x3e, 0x02, 0x04, 0x08, 0x10, 0x10, 0x10, 
-0x38, 0x1c, 0x22, 0x22, 0x1c, 0x22, 0x22, 0x1c, 
-0x39, 0x1c, 0x22, 0x22, 0x1e, 0x02, 0x04, 0x18, 
-0x44, 0x3c, 0x22, 0x22, 0x22, 0x22, 0x22, 0x3c, 
-0x45, 0x3e, 0x20, 0x20, 0x3c, 0x20, 0x20, 0x3e, 
-0x49, 0x1c, 0x08, 0x08, 0x08, 0x08, 0x08, 0x1c, 
-0x4b, 0x22, 0x24, 0x28, 0x30, 0x28, 0x24, 0x22, 
-0x4d, 0x22, 0x36, 0x2a, 0x2a, 0x22, 0x22, 0x22, 
-0x4f, 0x1c, 0x22, 0x22, 0x22, 0x22, 0x22, 0x1c, 
-0x52, 0x3c, 0x22, 0x22, 0x3c, 0x28, 0x24, 0x22, 
-0x54, 0x3e, 0x08, 0x08, 0x08, 0x08, 0x08, 0x08, 
-0x58, 0x22, 0x22, 0x14, 0x08, 0x14, 0x22, 0x22, 
-0x61, 0x00, 0x00, 0x1c, 0x02, 0x1e, 0x22, 0x1e, 
-0x63, 0x00, 0x00, 0x1c, 0x20, 0x20, 0x22, 0x1c, 
-0x65, 0x00, 0x00, 0x1c, 0x22, 0x3e, 0x20, 0x1c, 
-0x66, 0x0c, 0x12, 0x10, 0x38, 0x10, 0x10, 0x10, 
-0x67, 0x00, 0x1e, 0x22, 0x22, 0x1e, 0x02, 0x1c, 
-0x69, 0x08, 0x00, 0x18, 0x08, 0x08, 0x08, 0x1c, 
-0x6c, 0x18, 0x08, 0x08, 0x08, 0x08, 0x08, 0x1c, 
-0x6d, 0x00, 0x00, 0x34, 0x2a, 0x2a, 0x22, 0x22, 
-0x6e, 0x00, 0x00, 0x2c, 0x32, 0x22, 0x22, 0x22, 
-0x6f, 0x00, 0x00, 0x1c, 0x22, 0x22, 0x22, 0x1c, 
-0x70, 0x00, 0x00, 0x3c, 0x22, 0x3c, 0x20, 0x20, 
-0x72, 0x00, 0x00, 0x2c, 0x32, 0x20, 0x20, 0x20, 
-0x73, 0x00, 0x00, 0x1c, 0x20, 0x1c, 0x02, 0x3c, 
-0x74, 0x10, 0x10, 0x38, 0x10, 0x10, 0x12, 0x0c, 
-0x75, 0x00, 0x00, 0x22, 0x22, 0x22, 0x26, 0x1a, 
-0x99, 0x72, 
 };
-static unsigned int romDumpSize85 = 1218;
+static unsigned int romDumpSize85 = 0;
--- a/src/rom86.h
+++ b/src/rom86.h
@@ -1,89 +1,3 @@
 static unsigned char romDump86[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x38, 0x36, 0x2a, 0x2a, 
-0x1a, 0x0a, 0x00, 0x43, 0x72, 0x65, 0x61, 0x74, 
-0x65, 0x64, 0x20, 0x62, 0x79, 0x20, 0x74, 0x69, 
-0x70, 0x61, 0x63, 0x6b, 0x20, 0x30, 0x2e, 0x31, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x73, 0x02, 0x0c, 
-0x00, 0x63, 0x02, 0x12, 0x07, 0x52, 0x4f, 0x4d, 
-0x44, 0x75, 0x6d, 0x70, 0x20, 0x63, 0x02, 0x61, 
-0x02, 0x8e, 0x28, 0xfd, 0xcb, 0x15, 0xbe, 0xfd, 
-0xcb, 0x08, 0x9e, 0xdb, 0x05, 0x32, 0x00, 0x90, 
-0x21, 0x00, 0x00, 0x22, 0x03, 0x90, 0x22, 0x05, 
-0x90, 0xcd, 0xb1, 0x4a, 0xcd, 0x82, 0x4a, 0x21, 
-0x00, 0x00, 0x22, 0x0f, 0xc0, 0x21, 0x71, 0xd9, 
-0xcd, 0x37, 0x4a, 0x11, 0x04, 0x00, 0xed, 0x53, 
-0x0f, 0xc0, 0xcd, 0x37, 0x4a, 0xfb, 0x76, 0xcd, 
-0x68, 0x40, 0xfe, 0x37, 0x28, 0x2f, 0xcd, 0x09, 
-0xd9, 0x28, 0xf2, 0xcd, 0x69, 0xd8, 0x38, 0x53, 
-0xed, 0x5b, 0x13, 0x90, 0x21, 0x55, 0xaa, 0xb7, 
-0xed, 0x52, 0x28, 0x09, 0x7a, 0xb7, 0x20, 0x43, 
-0x43, 0x10, 0x0a, 0x18, 0xd8, 0x21, 0x01, 0x00, 
-0xcd, 0x9f, 0xd8, 0x18, 0xd0, 0x10, 0x10, 0x21, 
-0x02, 0x00, 0xcd, 0x9f, 0xd8, 0xcd, 0x82, 0x4a, 
-0x21, 0x00, 0x00, 0x22, 0x0f, 0xc0, 0xc9, 0x10, 
-0x20, 0x21, 0xa3, 0xd9, 0x22, 0x0b, 0x90, 0x21, 
-0x04, 0x00, 0x22, 0x09, 0x90, 0x21, 0x03, 0x00, 
-0xcd, 0xa7, 0xd8, 0x21, 0x02, 0x04, 0x22, 0x0f, 
-0xc0, 0x21, 0x99, 0xd9, 0xcd, 0x37, 0x4a, 0x18, 
-0x9c, 0x10, 0x1c, 0x21, 0x07, 0x00, 0x22, 0x0f, 
-0xc0, 0x21, 0x91, 0xd9, 0xcd, 0x37, 0x4a, 0x2a, 
-0x03, 0x90, 0x23, 0x22, 0x03, 0x90, 0xaf, 0xcd, 
-0x33, 0x4a, 0xcd, 0xfb, 0xd8, 0x18, 0xe0, 0x10, 
-0xe2, 0x2a, 0x17, 0x90, 0x3a, 0x19, 0x90, 0xcb, 
-0x24, 0x17, 0xcb, 0x24, 0x17, 0xd3, 0x05, 0x37, 
-0xcb, 0x1c, 0xcb, 0x3c, 0x22, 0x0b, 0x90, 0xcd, 
-0x5f, 0xd8, 0x20, 0x19, 0x67, 0x6f, 0x22, 0x0f, 
-0x90, 0x21, 0x00, 0x04, 0x22, 0x0d, 0x90, 0x21, 
-0x0d, 0x90, 0x22, 0x0b, 0x90, 0x21, 0x04, 0x00, 
-0x11, 0x07, 0x00, 0x18, 0x06, 0x21, 0x00, 0x04, 
-0x11, 0x06, 0x00, 0x22, 0x09, 0x90, 0xeb, 0xcd, 
-0xa7, 0xd8, 0x3a, 0x00, 0x90, 0xd3, 0x05, 0x38, 
-0x9a, 0x21, 0x02, 0x00, 0x22, 0x0f, 0xc0, 0x2a, 
-0x05, 0x90, 0x23, 0x22, 0x05, 0x90, 0xaf, 0xcd, 
-0x33, 0x4a, 0x3e, 0x4b, 0xcd, 0x2b, 0x4a, 0x18, 
-0x9c, 0xe9, 0x7e, 0x01, 0x00, 0x04, 0xed, 0xa1, 
-0xe0, 0x28, 0xfb, 0xc9, 0x21, 0x00, 0x00, 0x11, 
-0x13, 0x90, 0x06, 0x04, 0xcd, 0xe4, 0xd8, 0xd8, 
-0x3a, 0x16, 0x90, 0xc6, 0xff, 0xd8, 0x3a, 0x15, 
-0x90, 0xb7, 0x28, 0x0b, 0x47, 0xc6, 0xbf, 0xd8, 
-0x11, 0x17, 0x90, 0xcd, 0xe4, 0xd8, 0xd8, 0xe5, 
-0x11, 0x11, 0x90, 0x06, 0x02, 0xcd, 0xe4, 0xd8, 
-0xd1, 0xd8, 0x2a, 0x11, 0x90, 0xed, 0x52, 0xc8, 
-0x37, 0xc9, 0xe5, 0x21, 0x00, 0x00, 0x22, 0x09, 
-0x90, 0xe1, 0x22, 0x07, 0x90, 0x21, 0x00, 0x00, 
-0x11, 0x07, 0x90, 0x01, 0x04, 0x00, 0xcd, 0xce, 
-0xd8, 0xd8, 0xed, 0x5b, 0x0b, 0x90, 0xed, 0x4b, 
-0x09, 0x90, 0xcd, 0xce, 0xd8, 0xd8, 0x22, 0x11, 
-0x90, 0x11, 0x11, 0x90, 0x01, 0x02, 0x00, 0x18, 
-0x00, 0x78, 0xb1, 0xc8, 0x0b, 0xc5, 0x1a, 0x13, 
-0x4f, 0x06, 0x00, 0x09, 0xd5, 0xe5, 0xcd, 0x45, 
-0xd9, 0xe1, 0xd1, 0xc1, 0xd8, 0x18, 0xea, 0xc5, 
-0xe5, 0xd5, 0xcd, 0x0f, 0xd9, 0xd1, 0xe1, 0x38, 
-0x0b, 0x06, 0x00, 0x09, 0x79, 0x12, 0x13, 0xc1, 
-0x10, 0xed, 0xb7, 0xc5, 0xc1, 0xc9, 0x06, 0xff, 
-0xf5, 0x3e, 0xc0, 0xd3, 0x07, 0xfb, 0x76, 0x76, 
-0x10, 0xfb, 0xf1, 0xc9, 0xdb, 0x07, 0x2f, 0xe6, 
-0x03, 0xc9, 0xf3, 0x3e, 0xc0, 0xd3, 0x07, 0x06, 
-0x08, 0x16, 0xff, 0xcd, 0x35, 0xd9, 0xd8, 0xc6, 
-0xfe, 0x3e, 0xe8, 0x30, 0x03, 0xc0, 0x3e, 0xd4, 
-0xd3, 0x07, 0xcb, 0x19, 0x14, 0xcd, 0x35, 0xd9, 
-0xd8, 0x3e, 0xc0, 0xd3, 0x07, 0x10, 0xe2, 0xc9, 
-0x21, 0xff, 0xff, 0x2b, 0xdb, 0x07, 0xaa, 0xe6, 
-0x03, 0xc0, 0x7c, 0xb5, 0x20, 0xf5, 0x37, 0xc9, 
-0xf3, 0x06, 0x08, 0xcd, 0x5b, 0xd9, 0xd8, 0xcb, 
-0x19, 0x3e, 0xd4, 0x30, 0x02, 0x3e, 0xe8, 0x14, 
-0xcd, 0x5f, 0xd9, 0xd8, 0x10, 0xed, 0x3e, 0xc0, 
-0x16, 0xff, 0xd3, 0x07, 0x21, 0xff, 0xff, 0x2b, 
-0xdb, 0x07, 0xaa, 0xe6, 0x03, 0xc8, 0x7c, 0xb5, 
-0x20, 0xf5, 0x37, 0xc9, 0x44, 0x75, 0x6d, 0x70, 
-0x69, 0x6e, 0x67, 0x20, 0x52, 0x4f, 0x4d, 0x2e, 
-0x2e, 0x2e, 0x00, 0x5b, 0x45, 0x58, 0x49, 0x54, 
-0x5d, 0x20, 0x74, 0x6f, 0x20, 0x63, 0x61, 0x6e, 
-0x63, 0x65, 0x6c, 0x00, 0x45, 0x72, 0x72, 0x6f, 
-0x72, 0x73, 0x3a, 0x00, 0x30, 0x4b, 0x20, 0x6f, 
-0x66, 0x20, 0x32, 0x35, 0x36, 0x4b, 0x00, 0x00, 
-0x04, 0x00, 0x5d, 0xf9, 
 };
-static unsigned int romDumpSize86 = 684;
+static unsigned int romDumpSize86 = 0;
--- a/src/rom89.h
+++ b/src/rom89.h
@@ -1,183 +1,3 @@
 static unsigned char romDump89[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x38, 0x39, 0x2a, 0x2a, 
-0x01, 0x00, 0x6d, 0x61, 0x69, 0x6e, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x01, 0x00, 0x52, 0x00, 0x00, 0x00, 
-0x72, 0x6f, 0x6d, 0x64, 0x75, 0x6d, 0x70, 0x00, 
-0x21, 0x00, 0x00, 0x00, 0x9d, 0x05, 0x00, 0x00, 
-0xa5, 0x5a, 0x00, 0x00, 0x00, 0x00, 0x05, 0x43, 
-0x20, 0x78, 0x00, 0xc8, 0x70, 0x01, 0x22, 0x08, 
-0x02, 0x81, 0x00, 0x40, 0x00, 0x00, 0x66, 0x10, 
-0x42, 0x40, 0x22, 0x68, 0x00, 0xbc, 0x0c, 0x29, 
-0x00, 0xc8, 0x00, 0x02, 0x65, 0x02, 0x70, 0x03, 
-0x43, 0xfa, 0x00, 0xac, 0x32, 0x80, 0x48, 0xe7, 
-0x1c, 0x04, 0x2a, 0x78, 0x00, 0xc8, 0x43, 0xfa, 
-0x00, 0x00, 0x28, 0x09, 0x02, 0x84, 0x00, 0x03, 
-0xff, 0xff, 0x76, 0x01, 0x61, 0x3e, 0x4a, 0x40, 
-0x66, 0x14, 0x28, 0x2f, 0x00, 0x10, 0x0c, 0x84, 
-0x00, 0x1f, 0xff, 0xff, 0x62, 0x08, 0x02, 0x84, 
-0x00, 0x03, 0xff, 0xff, 0x61, 0x26, 0x30, 0x3c, 
-0x03, 0xbf, 0x41, 0xf8, 0x4c, 0x00, 0x2f, 0x18, 
-0x51, 0xc8, 0xff, 0xfc, 0x61, 0x6a, 0x30, 0x3c, 
-0x03, 0xbf, 0x41, 0xf8, 0x5b, 0x00, 0x21, 0x1f, 
-0x51, 0xc8, 0xff, 0xfc, 0x61, 0x06, 0x4c, 0xdf, 
-0x20, 0x38, 0x4e, 0x75, 0x3f, 0x3c, 0x00, 0x06, 
-0x42, 0xa7, 0x20, 0x6d, 0x01, 0xb0, 0x4e, 0x90, 
-0x5c, 0x8f, 0x42, 0x40, 0x2a, 0x08, 0x67, 0x3c, 
-0x3f, 0x28, 0x00, 0x0c, 0x20, 0x6d, 0x02, 0x58, 
-0x4e, 0x90, 0x54, 0x8f, 0xb1, 0xc4, 0x62, 0x20, 
-0x70, 0x00, 0x30, 0x10, 0xd1, 0xc0, 0xb1, 0xc4, 
-0x63, 0x16, 0x70, 0x01, 0x20, 0x45, 0x41, 0xe8, 
-0x00, 0x0b, 0x4a, 0x43, 0x67, 0x12, 0x72, 0x10, 
-0x16, 0x10, 0xc6, 0x41, 0x83, 0x10, 0x4e, 0x75, 
-0x20, 0x6d, 0x01, 0xb4, 0x4e, 0x90, 0x60, 0xc2, 
-0x02, 0x10, 0x00, 0xef, 0x4e, 0x75, 0xff, 0xff, 
-0x4f, 0xef, 0xef, 0xb4, 0x48, 0xe7, 0x1e, 0x30, 
-0x20, 0x6d, 0x06, 0x78, 0x4e, 0x90, 0x3f, 0x3c, 
-0x00, 0x02, 0x20, 0x6d, 0x06, 0x3c, 0x4e, 0x90, 
-0x48, 0x7a, 0x03, 0xb8, 0x48, 0x7a, 0x03, 0xb9, 
-0x45, 0xef, 0x10, 0x48, 0x2f, 0x0a, 0x20, 0x6d, 
-0x01, 0x4c, 0x4e, 0x90, 0x3f, 0x3c, 0x00, 0x01, 
-0x2f, 0x0a, 0x42, 0xa7, 0x20, 0x6d, 0x06, 0xa4, 
-0x4e, 0x90, 0x61, 0x00, 0x03, 0xf8, 0x3f, 0x00, 
-0x48, 0x7a, 0x03, 0xa3, 0x2f, 0x0a, 0x20, 0x6d, 
-0x01, 0x4c, 0x4e, 0x90, 0x4f, 0xef, 0x00, 0x20, 
-0x3e, 0xbc, 0x00, 0x01, 0x2f, 0x0a, 0x48, 0x78, 
-0x00, 0x14, 0x20, 0x6d, 0x06, 0xa4, 0x4e, 0x90, 
-0x22, 0x0d, 0x02, 0x81, 0x00, 0xe0, 0x00, 0x00, 
-0x2f, 0x01, 0x48, 0x7a, 0x03, 0x84, 0x2f, 0x0a, 
-0x20, 0x6d, 0x01, 0x4c, 0x4e, 0x90, 0x3f, 0x3c, 
-0x00, 0x01, 0x2f, 0x0a, 0x48, 0x78, 0x00, 0x28, 
-0x20, 0x6d, 0x06, 0xa4, 0x4e, 0x90, 0x4f, 0xef, 
-0x00, 0x1c, 0x2e, 0xbc, 0x00, 0x00, 0x00, 0x00, 
-0x2f, 0x0a, 0x20, 0x6d, 0x01, 0x4c, 0x4e, 0x90, 
-0x3f, 0x3c, 0x00, 0x01, 0x2f, 0x0a, 0x48, 0x78, 
-0x00, 0x50, 0x20, 0x6d, 0x06, 0xa4, 0x4e, 0x90, 
-0x3c, 0x3a, 0xff, 0x54, 0x4f, 0xef, 0x00, 0x12, 
-0x48, 0x78, 0x00, 0x14, 0x48, 0x78, 0x00, 0x04, 
-0x45, 0xef, 0x00, 0x22, 0x2f, 0x0a, 0x20, 0x6d, 
-0x01, 0x6c, 0x4e, 0x90, 0x4f, 0xef, 0x00, 0x0c, 
-0x4a, 0x40, 0x67, 0x04, 0x78, 0xff, 0x60, 0x60, 
-0x18, 0x2f, 0x00, 0x1b, 0xe1, 0x4c, 0x42, 0x40, 
-0x10, 0x2f, 0x00, 0x1a, 0x88, 0x40, 0x16, 0x2f, 
-0x00, 0x1d, 0xe1, 0x4b, 0x42, 0x40, 0x10, 0x2f, 
-0x00, 0x1c, 0x86, 0x40, 0x47, 0xfa, 0x02, 0xc4, 
-0x70, 0x04, 0x20, 0x4a, 0x4e, 0x93, 0x48, 0x78, 
-0x00, 0x14, 0x7a, 0x00, 0x3a, 0x03, 0x2f, 0x05, 
-0x2f, 0x0a, 0x20, 0x6d, 0x01, 0x6c, 0x4e, 0x90, 
-0x4f, 0xef, 0x00, 0x0c, 0x4a, 0x40, 0x66, 0x20, 
-0x48, 0x78, 0x00, 0x14, 0x48, 0x78, 0x00, 0x02, 
-0x48, 0x72, 0x58, 0x00, 0x20, 0x6d, 0x01, 0x6c, 
-0x4e, 0x90, 0x4f, 0xef, 0x00, 0x0c, 0x4a, 0x40, 
-0x66, 0x06, 0x30, 0x03, 0x20, 0x4a, 0x4e, 0x93, 
-0x20, 0x6d, 0x01, 0x48, 0x4e, 0x90, 0x3a, 0x00, 
-0x0c, 0x44, 0x00, 0x02, 0x67, 0x20, 0x0c, 0x44, 
-0x00, 0x03, 0x67, 0x2c, 0x0c, 0x44, 0x00, 0x05, 
-0x67, 0x00, 0x00, 0xdc, 0x0c, 0x44, 0xaa, 0x55, 
-0x66, 0x00, 0x01, 0xe0, 0x93, 0xc9, 0x42, 0x41, 
-0x70, 0x01, 0x60, 0x00, 0x01, 0xd0, 0x93, 0xc9, 
-0x42, 0x41, 0x70, 0x02, 0x41, 0xef, 0x00, 0x1a, 
-0x61, 0x00, 0x01, 0xd8, 0x60, 0x00, 0x01, 0xca, 
-0x0c, 0x46, 0x00, 0x03, 0x66, 0x04, 0x70, 0x01, 
-0x60, 0x14, 0x20, 0x0d, 0x02, 0x80, 0x00, 0xe0, 
-0x00, 0x00, 0x0c, 0x80, 0x00, 0x80, 0x00, 0x00, 
-0x57, 0xc0, 0x48, 0x80, 0x44, 0x40, 0x48, 0xc0, 
-0x72, 0x20, 0x48, 0x41, 0xe1, 0xa9, 0x24, 0x01, 
-0x42, 0x42, 0x48, 0x42, 0xe0, 0x4a, 0x20, 0x01, 
-0x02, 0x80, 0x00, 0xff, 0x00, 0x00, 0xe0, 0x88, 
-0x84, 0x80, 0x20, 0x01, 0x02, 0x80, 0x00, 0x00, 
-0xff, 0x00, 0xe1, 0x88, 0xe1, 0x49, 0x48, 0x41, 
-0x42, 0x41, 0x82, 0x80, 0x43, 0xef, 0x10, 0x64, 
-0x84, 0x81, 0x23, 0x02, 0x72, 0x04, 0x70, 0x03, 
-0x41, 0xef, 0x00, 0x1a, 0x61, 0x00, 0x01, 0x74, 
-0x20, 0x0d, 0x20, 0x6d, 0x01, 0x4c, 0x0c, 0x46, 
-0x00, 0x03, 0x66, 0x04, 0x70, 0x01, 0x60, 0x12, 
-0x02, 0x80, 0x00, 0xe0, 0x00, 0x00, 0x0c, 0x80, 
-0x00, 0x80, 0x00, 0x00, 0x57, 0xc0, 0x48, 0x80, 
-0x44, 0x40, 0x32, 0x00, 0x48, 0xc1, 0x70, 0x20, 
-0x48, 0x40, 0xe3, 0xa8, 0x72, 0x0a, 0xe2, 0xa8, 
-0x2f, 0x00, 0x48, 0x7a, 0x02, 0x0d, 0x45, 0xef, 
-0x10, 0x28, 0x2f, 0x0a, 0x4e, 0x90, 0x3f, 0x3c, 
-0x00, 0x04, 0x2f, 0x0a, 0x48, 0x78, 0x00, 0x3c, 
-0x20, 0x6d, 0x06, 0xa4, 0x4e, 0x90, 0x4f, 0xef, 
-0x00, 0x16, 0x60, 0x00, 0x01, 0x0e, 0x76, 0x00, 
-0x16, 0x2f, 0x00, 0x1b, 0xe1, 0x8b, 0x86, 0x2f, 
-0x00, 0x1a, 0x12, 0x2f, 0x00, 0x1d, 0xe1, 0x49, 
-0x48, 0x41, 0x42, 0x41, 0x70, 0x00, 0x10, 0x2f, 
-0x00, 0x1c, 0x48, 0x40, 0x42, 0x40, 0x82, 0x80, 
-0x86, 0x81, 0x0c, 0x46, 0x00, 0x03, 0x66, 0x04, 
-0x70, 0x01, 0x60, 0x14, 0x20, 0x0d, 0x02, 0x80, 
-0x00, 0xe0, 0x00, 0x00, 0x0c, 0x80, 0x00, 0x80, 
-0x00, 0x00, 0x57, 0xc0, 0x48, 0x80, 0x44, 0x40, 
-0x32, 0x00, 0x48, 0xc1, 0x70, 0x20, 0x48, 0x40, 
-0xe3, 0xa8, 0xb0, 0x83, 0x64, 0x0e, 0x93, 0xc9, 
-0x42, 0x41, 0x70, 0x04, 0x41, 0xef, 0x00, 0x1a, 
-0x61, 0x00, 0x00, 0xc0, 0x20, 0x0d, 0x20, 0x6d, 
-0x01, 0x4c, 0x0c, 0x46, 0x00, 0x03, 0x66, 0x04, 
-0x70, 0x01, 0x60, 0x12, 0x02, 0x80, 0x00, 0xe0, 
-0x00, 0x00, 0x0c, 0x80, 0x00, 0x80, 0x00, 0x00, 
-0x57, 0xc0, 0x48, 0x80, 0x44, 0x40, 0x32, 0x00, 
-0x48, 0xc1, 0x70, 0x20, 0x48, 0x40, 0xe3, 0xa8, 
-0x72, 0x0a, 0xe2, 0xa8, 0x2f, 0x00, 0x20, 0x03, 
-0xe2, 0xa8, 0x2f, 0x00, 0x48, 0x7a, 0x01, 0x60, 
-0x45, 0xef, 0x10, 0x2c, 0x2f, 0x0a, 0x4e, 0x90, 
-0x3f, 0x3c, 0x00, 0x04, 0x2f, 0x0a, 0x48, 0x78, 
-0x00, 0x3c, 0x20, 0x6d, 0x06, 0xa4, 0x4e, 0x90, 
-0x20, 0x0d, 0x02, 0x80, 0x00, 0xe0, 0x00, 0x00, 
-0x22, 0x40, 0xd3, 0xc3, 0x10, 0x11, 0x72, 0x00, 
-0x4f, 0xef, 0x00, 0x1a, 0xb0, 0x31, 0x18, 0x00, 
-0x66, 0x34, 0x34, 0x01, 0x52, 0x42, 0x52, 0x81, 
-0x0c, 0x81, 0x00, 0x00, 0x10, 0x00, 0x66, 0xec, 
-0x0c, 0x42, 0x10, 0x00, 0x66, 0x20, 0x02, 0x40, 
-0x00, 0xff, 0x42, 0x2f, 0x10, 0x5c, 0x1f, 0x7c, 
-0x00, 0x10, 0x10, 0x5d, 0x1f, 0x40, 0x10, 0x5e, 
-0x42, 0x2f, 0x10, 0x5f, 0x43, 0xef, 0x10, 0x5c, 
-0x72, 0x04, 0x70, 0x07, 0x60, 0x06, 0x32, 0x3c, 
-0x10, 0x00, 0x70, 0x06, 0x41, 0xef, 0x00, 0x1a, 
-0x61, 0x10, 0x4a, 0x45, 0x67, 0x00, 0xfd, 0x7a, 
-0x4c, 0xdf, 0x0c, 0x78, 0x4f, 0xef, 0x10, 0x4c, 
-0x4e, 0x75, 0x48, 0xe7, 0x1c, 0x38, 0x26, 0x48, 
-0x38, 0x01, 0x2a, 0x09, 0x28, 0x48, 0x18, 0xc0, 
-0xe0, 0x48, 0x18, 0x80, 0x17, 0x41, 0x00, 0x02, 
-0x30, 0x01, 0xe0, 0x48, 0x11, 0x40, 0x00, 0x03, 
-0x48, 0x78, 0x00, 0x04, 0x2f, 0x08, 0x20, 0x6d, 
-0x01, 0x68, 0x4e, 0x90, 0x50, 0x8f, 0x4a, 0x40, 
-0x66, 0x3a, 0x3f, 0x04, 0x42, 0x67, 0x2f, 0x05, 
-0x20, 0x6d, 0x01, 0x68, 0x4e, 0x90, 0x50, 0x8f, 
-0x4a, 0x40, 0x66, 0x28, 0x45, 0xfa, 0x00, 0x2c, 
-0x70, 0x04, 0x20, 0x4b, 0x4e, 0x92, 0x36, 0x00, 
-0x30, 0x04, 0x20, 0x45, 0x4e, 0x92, 0xd6, 0x40, 
-0x16, 0x83, 0xe0, 0x4b, 0x18, 0x83, 0x48, 0x78, 
-0x00, 0x02, 0x2f, 0x0b, 0x20, 0x6d, 0x01, 0x68, 
-0x4e, 0x90, 0x50, 0x8f, 0x4c, 0xdf, 0x1c, 0x38, 
-0x4e, 0x75, 0x34, 0x00, 0x42, 0x41, 0x93, 0xc9, 
-0x60, 0x10, 0x70, 0x00, 0x30, 0x01, 0x10, 0x30, 
-0x08, 0x00, 0x02, 0x40, 0x00, 0xff, 0xd2, 0xc0, 
-0x52, 0x41, 0xb4, 0x41, 0x66, 0xec, 0x30, 0x09, 
-0x4e, 0x75, 0x31, 0x2e, 0x30, 0x32, 0x00, 0x52, 
-0x6f, 0x6d, 0x44, 0x75, 0x6d, 0x70, 0x65, 0x72, 
-0x20, 0x76, 0x25, 0x73, 0x00, 0x54, 0x79, 0x70, 
-0x65, 0x3a, 0x20, 0x48, 0x57, 0x25, 0x69, 0x00, 
-0x52, 0x4f, 0x4d, 0x20, 0x62, 0x61, 0x73, 0x65, 
-0x3a, 0x20, 0x30, 0x78, 0x25, 0x6c, 0x78, 0x00, 
-0x62, 0x79, 0x20, 0x54, 0x68, 0x65, 0x20, 0x54, 
-0x69, 0x4c, 0x50, 0x20, 0x54, 0x65, 0x61, 0x6d, 
-0x00, 0x53, 0x69, 0x7a, 0x65, 0x3a, 0x20, 0x25, 
-0x6c, 0x75, 0x20, 0x4b, 0x42, 0x00, 0x44, 0x6f, 
-0x6e, 0x65, 0x3a, 0x20, 0x25, 0x6c, 0x75, 0x2f, 
-0x25, 0x6c, 0x75, 0x4b, 0x42, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x00, 0x00, 0x20, 0x38, 0x00, 0xc8, 
-0x02, 0x80, 0x00, 0xe0, 0x00, 0x00, 0x20, 0x40, 
-0x22, 0x68, 0x01, 0x04, 0xd1, 0xfc, 0x00, 0x01, 
-0x00, 0x00, 0xb3, 0xc8, 0x64, 0x14, 0x0c, 0x51, 
-0x00, 0x16, 0x63, 0x0e, 0x4e, 0x4c, 0x46, 0xfc, 
-0x30, 0x00, 0x40, 0xc1, 0x46, 0xc0, 0xe7, 0x49, 
-0x6a, 0x04, 0x70, 0x01, 0x4e, 0x75, 0x20, 0x29, 
-0x00, 0x16, 0x4e, 0x75, 0x00, 0x00, 0x04, 0xd0, 
-0x01, 0x5c, 0xf3, 0x59, 0x81, 
 };
-static unsigned int romDumpSize89 = 1437;
+static unsigned int romDumpSize89 = 0;
--- a/src/rom89t.h
+++ b/src/rom89t.h
@@ -1,220 +1,3 @@
 static unsigned char romDump89t[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x38, 0x39, 0x2a, 0x2a, 
-0x01, 0x00, 0x6d, 0x61, 0x69, 0x6e, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x01, 0x00, 0x52, 0x00, 0x00, 0x00, 
-0x72, 0x6f, 0x6d, 0x64, 0x75, 0x6d, 0x70, 0x00, 
-0x21, 0x00, 0x00, 0x00, 0xc5, 0x06, 0x00, 0x00, 
-0xa5, 0x5a, 0x00, 0x00, 0x00, 0x00, 0x06, 0x6b, 
-0x20, 0x78, 0x00, 0xc8, 0x70, 0x01, 0x22, 0x08, 
-0x02, 0x81, 0x00, 0x40, 0x00, 0x00, 0x66, 0x10, 
-0x42, 0x40, 0x22, 0x68, 0x00, 0xbc, 0x0c, 0x29, 
-0x00, 0xc8, 0x00, 0x02, 0x65, 0x02, 0x70, 0x03, 
-0x4a, 0x40, 0x67, 0x0e, 0x48, 0x7a, 0x00, 0xb6, 
-0x20, 0x68, 0x03, 0x98, 0x4e, 0x90, 0x58, 0x8f, 
-0x4e, 0x75, 0x48, 0xe7, 0x1c, 0x04, 0x2a, 0x78, 
-0x00, 0xc8, 0x43, 0xfa, 0x00, 0x00, 0x28, 0x09, 
-0x02, 0x84, 0x00, 0x03, 0xff, 0xff, 0x76, 0x01, 
-0x61, 0x40, 0x4a, 0x40, 0x66, 0x14, 0x28, 0x2f, 
-0x00, 0x10, 0x0c, 0x84, 0x00, 0x1f, 0xff, 0xff, 
-0x62, 0x08, 0x02, 0x84, 0x00, 0x03, 0xff, 0xff, 
-0x61, 0x28, 0x30, 0x3c, 0x03, 0xbf, 0x41, 0xf8, 
-0x4c, 0x00, 0x2f, 0x18, 0x51, 0xc8, 0xff, 0xfc, 
-0x61, 0x00, 0x00, 0x82, 0x30, 0x3c, 0x03, 0xbf, 
-0x41, 0xf8, 0x5b, 0x00, 0x21, 0x1f, 0x51, 0xc8, 
-0xff, 0xfc, 0x61, 0x06, 0x4c, 0xdf, 0x20, 0x38, 
-0x4e, 0x75, 0x3f, 0x3c, 0x00, 0x06, 0x42, 0xa7, 
-0x20, 0x6d, 0x01, 0xb0, 0x4e, 0x90, 0x5c, 0x8f, 
-0x42, 0x40, 0x2a, 0x08, 0x67, 0x3c, 0x3f, 0x28, 
-0x00, 0x0c, 0x20, 0x6d, 0x02, 0x58, 0x4e, 0x90, 
-0x54, 0x8f, 0xb1, 0xc4, 0x62, 0x20, 0x70, 0x00, 
-0x30, 0x10, 0xd1, 0xc0, 0xb1, 0xc4, 0x63, 0x16, 
-0x70, 0x01, 0x20, 0x45, 0x41, 0xe8, 0x00, 0x0b, 
-0x4a, 0x43, 0x67, 0x12, 0x72, 0x10, 0x16, 0x10, 
-0xc6, 0x41, 0x83, 0x10, 0x4e, 0x75, 0x20, 0x6d, 
-0x01, 0xb4, 0x4e, 0x90, 0x60, 0xc2, 0x02, 0x10, 
-0x00, 0xef, 0x4e, 0x75, 0x57, 0x72, 0x6f, 0x6e, 
-0x67, 0x20, 0x63, 0x61, 0x6c, 0x63, 0x75, 0x6c, 
-0x61, 0x74, 0x6f, 0x72, 0x20, 0x6d, 0x6f, 0x64, 
-0x65, 0x6c, 0x00, 0x00, 0x4e, 0x56, 0xef, 0xb4, 
-0x48, 0xe7, 0x1f, 0x20, 0x20, 0x6d, 0x06, 0x78, 
-0x4e, 0x90, 0x3f, 0x3c, 0x00, 0x02, 0x20, 0x6d, 
-0x06, 0x3c, 0x4e, 0x90, 0x48, 0x7a, 0x04, 0xb8, 
-0x48, 0x7a, 0x04, 0xbd, 0x45, 0xee, 0xff, 0xda, 
-0x2f, 0x0a, 0x20, 0x6d, 0x01, 0x4c, 0x4e, 0x90, 
-0x3f, 0x3c, 0x00, 0x01, 0x2f, 0x0a, 0x42, 0xa7, 
-0x20, 0x6d, 0x06, 0xa4, 0x4e, 0x90, 0x61, 0x00, 
-0x04, 0xfc, 0x3f, 0x00, 0x48, 0x7a, 0x04, 0xa7, 
-0x2f, 0x0a, 0x20, 0x6d, 0x01, 0x4c, 0x4e, 0x90, 
-0x4f, 0xef, 0x00, 0x20, 0x3e, 0xbc, 0x00, 0x01, 
-0x2f, 0x0a, 0x48, 0x78, 0x00, 0x14, 0x20, 0x6d, 
-0x06, 0xa4, 0x4e, 0x90, 0x22, 0x0d, 0x02, 0x81, 
-0x00, 0xe0, 0x00, 0x00, 0x2f, 0x01, 0x48, 0x7a, 
-0x04, 0x88, 0x2f, 0x0a, 0x20, 0x6d, 0x01, 0x4c, 
-0x4e, 0x90, 0x3f, 0x3c, 0x00, 0x01, 0x2f, 0x0a, 
-0x48, 0x78, 0x00, 0x28, 0x20, 0x6d, 0x06, 0xa4, 
-0x4e, 0x90, 0x4f, 0xef, 0x00, 0x1c, 0x2e, 0xbc, 
-0x00, 0x00, 0x00, 0x00, 0x2f, 0x0a, 0x20, 0x6d, 
-0x01, 0x4c, 0x4e, 0x90, 0x3f, 0x3c, 0x00, 0x01, 
-0x2f, 0x0a, 0x48, 0x78, 0x00, 0x50, 0x20, 0x6d, 
-0x06, 0xa4, 0x4e, 0x90, 0x4f, 0xef, 0x00, 0x12, 
-0x20, 0x6d, 0x0f, 0x10, 0x20, 0x28, 0xfb, 0xa8, 
-0x22, 0x28, 0xfb, 0xac, 0x24, 0x3c, 0x76, 0x02, 
-0x10, 0x03, 0x26, 0x3c, 0x26, 0x1f, 0x4e, 0x75, 
-0x92, 0x83, 0x91, 0x82, 0x66, 0x06, 0x41, 0xe8, 
-0xfb, 0xc6, 0x60, 0x08, 0x30, 0x7c, 0xff, 0xaa, 
-0xd1, 0xed, 0x0a, 0x4c, 0x20, 0x50, 0x41, 0xe8, 
-0xff, 0x0a, 0x48, 0x78, 0x07, 0xd0, 0x48, 0x78, 
-0x00, 0x04, 0x2a, 0x0e, 0x06, 0x85, 0xff, 0xff, 
-0xef, 0xb6, 0x2f, 0x05, 0x4e, 0x90, 0x4f, 0xef, 
-0x00, 0x0c, 0x4a, 0x40, 0x67, 0x06, 0x78, 0xff, 
-0x60, 0x00, 0x00, 0xc4, 0x18, 0x2e, 0xef, 0xb7, 
-0xe1, 0x4c, 0x42, 0x40, 0x10, 0x2e, 0xef, 0xb6, 
-0x88, 0x40, 0x16, 0x2e, 0xef, 0xb9, 0xe1, 0x4b, 
-0x42, 0x40, 0x10, 0x2e, 0xef, 0xb8, 0x86, 0x40, 
-0x70, 0x04, 0x20, 0x45, 0x61, 0x00, 0x03, 0x90, 
-0x20, 0x6d, 0x0f, 0x10, 0x20, 0x28, 0xfb, 0xa8, 
-0x22, 0x28, 0xfb, 0xac, 0x2a, 0x3c, 0x76, 0x02, 
-0x10, 0x03, 0x2c, 0x3c, 0x26, 0x1f, 0x4e, 0x75, 
-0x92, 0x86, 0x91, 0x85, 0x66, 0x06, 0x41, 0xe8, 
-0xfb, 0xc6, 0x60, 0x08, 0x30, 0x7c, 0xff, 0xaa, 
-0xd1, 0xed, 0x0a, 0x4c, 0x7a, 0x00, 0x3a, 0x03, 
-0x20, 0x50, 0x41, 0xe8, 0xff, 0x0a, 0x48, 0x78, 
-0x07, 0xd0, 0x2f, 0x05, 0x48, 0x6e, 0xef, 0xb6, 
-0x4e, 0x90, 0x4f, 0xef, 0x00, 0x0c, 0x4a, 0x40, 
-0x66, 0x54, 0x20, 0x6d, 0x0f, 0x10, 0x20, 0x28, 
-0xfb, 0xa8, 0x22, 0x28, 0xfb, 0xac, 0x2c, 0x3c, 
-0x76, 0x02, 0x10, 0x03, 0x2e, 0x3c, 0x26, 0x1f, 
-0x4e, 0x75, 0x92, 0x87, 0x91, 0x86, 0x66, 0x06, 
-0x41, 0xe8, 0xfb, 0xc6, 0x60, 0x08, 0x30, 0x7c, 
-0xff, 0xaa, 0xd1, 0xed, 0x0a, 0x4c, 0x20, 0x50, 
-0x41, 0xe8, 0xff, 0x0a, 0x48, 0x78, 0x07, 0xd0, 
-0x48, 0x78, 0x00, 0x02, 0x45, 0xee, 0xef, 0xb6, 
-0x48, 0x72, 0x58, 0x00, 0x4e, 0x90, 0x4f, 0xef, 
-0x00, 0x0c, 0x4a, 0x40, 0x66, 0x08, 0x30, 0x03, 
-0x20, 0x4a, 0x61, 0x00, 0x02, 0xf2, 0x20, 0x6d, 
-0x01, 0x48, 0x4e, 0x90, 0x3a, 0x00, 0x0c, 0x44, 
-0x00, 0x02, 0x67, 0x20, 0x0c, 0x44, 0x00, 0x03, 
-0x67, 0x2c, 0x0c, 0x44, 0x00, 0x05, 0x67, 0x00, 
-0x00, 0x96, 0x0c, 0x44, 0xaa, 0x55, 0x66, 0x00, 
-0x01, 0x7c, 0x93, 0xc9, 0x42, 0x41, 0x70, 0x01, 
-0x60, 0x00, 0x01, 0x6c, 0x93, 0xc9, 0x42, 0x41, 
-0x70, 0x02, 0x41, 0xee, 0xef, 0xb6, 0x61, 0x00, 
-0x01, 0xa8, 0x60, 0x00, 0x01, 0x66, 0x20, 0x0d, 
-0x02, 0x80, 0x00, 0xe0, 0x00, 0x00, 0x72, 0x40, 
-0x48, 0x41, 0x0c, 0x80, 0x00, 0x80, 0x00, 0x00, 
-0x67, 0x04, 0x72, 0x20, 0x48, 0x41, 0xe0, 0x89, 
-0x22, 0x4e, 0x23, 0x01, 0x72, 0x04, 0x70, 0x03, 
-0x41, 0xee, 0xef, 0xb6, 0x61, 0x00, 0x01, 0x7a, 
-0x20, 0x0d, 0x20, 0x6d, 0x01, 0x4c, 0x02, 0x80, 
-0x00, 0xe0, 0x00, 0x00, 0x22, 0x3c, 0x00, 0x00, 
-0x10, 0x00, 0x0c, 0x80, 0x00, 0x80, 0x00, 0x00, 
-0x67, 0x06, 0x22, 0x3c, 0x00, 0x00, 0x08, 0x00, 
-0x2f, 0x01, 0x48, 0x7a, 0x02, 0xc5, 0x45, 0xee, 
-0xff, 0xbc, 0x2f, 0x0a, 0x4e, 0x90, 0x3f, 0x3c, 
-0x00, 0x04, 0x2f, 0x0a, 0x48, 0x78, 0x00, 0x3c, 
-0x20, 0x6d, 0x06, 0xa4, 0x4e, 0x90, 0x4f, 0xef, 
-0x00, 0x16, 0x60, 0x00, 0x00, 0xf0, 0x76, 0x00, 
-0x16, 0x2e, 0xef, 0xb7, 0xe1, 0x8b, 0x86, 0x2e, 
-0xef, 0xb6, 0x10, 0x2e, 0xef, 0xb9, 0xe1, 0x48, 
-0x48, 0x40, 0x42, 0x40, 0x72, 0x00, 0x12, 0x2e, 
-0xef, 0xb8, 0x48, 0x41, 0x42, 0x41, 0x80, 0x81, 
-0x86, 0x80, 0x20, 0x0d, 0x02, 0x80, 0x00, 0xe0, 
-0x00, 0x00, 0x72, 0x40, 0x48, 0x41, 0x0c, 0x80, 
-0x00, 0x80, 0x00, 0x00, 0x67, 0x04, 0x72, 0x20, 
-0x48, 0x41, 0xb2, 0x83, 0x64, 0x0e, 0x93, 0xc9, 
-0x42, 0x41, 0x70, 0x04, 0x41, 0xee, 0xef, 0xb6, 
-0x61, 0x00, 0x00, 0xe6, 0x20, 0x0d, 0x20, 0x6d, 
-0x01, 0x4c, 0x02, 0x80, 0x00, 0xe0, 0x00, 0x00, 
-0x22, 0x3c, 0x00, 0x00, 0x10, 0x00, 0x0c, 0x80, 
-0x00, 0x80, 0x00, 0x00, 0x67, 0x06, 0x22, 0x3c, 
-0x00, 0x00, 0x08, 0x00, 0x2f, 0x01, 0x20, 0x03, 
-0x7e, 0x0a, 0xee, 0xa8, 0x2f, 0x00, 0x48, 0x7a, 
-0x02, 0x36, 0x45, 0xee, 0xff, 0xbc, 0x2f, 0x0a, 
-0x4e, 0x90, 0x3f, 0x3c, 0x00, 0x04, 0x2f, 0x0a, 
-0x48, 0x78, 0x00, 0x3c, 0x20, 0x6d, 0x06, 0xa4, 
-0x4e, 0x90, 0x20, 0x0d, 0x02, 0x80, 0x00, 0xe0, 
-0x00, 0x00, 0x22, 0x40, 0xd3, 0xc3, 0x10, 0x11, 
-0x72, 0x00, 0x4f, 0xef, 0x00, 0x1a, 0xb0, 0x31, 
-0x18, 0x00, 0x66, 0x34, 0x34, 0x01, 0x52, 0x42, 
-0x52, 0x81, 0x0c, 0x81, 0x00, 0x00, 0x10, 0x00, 
-0x66, 0xec, 0x0c, 0x42, 0x10, 0x00, 0x66, 0x20, 
-0x02, 0x40, 0x00, 0xff, 0x42, 0x2e, 0xff, 0xf8, 
-0x1d, 0x7c, 0x00, 0x10, 0xff, 0xf9, 0x1d, 0x40, 
-0xff, 0xfa, 0x42, 0x2e, 0xff, 0xfb, 0x43, 0xee, 
-0xff, 0xf8, 0x72, 0x04, 0x70, 0x07, 0x60, 0x06, 
-0x32, 0x3c, 0x10, 0x00, 0x70, 0x06, 0x41, 0xee, 
-0xef, 0xb6, 0x61, 0x44, 0x4a, 0x45, 0x67, 0x00, 
-0xfd, 0x48, 0x20, 0x6d, 0x0f, 0x10, 0x20, 0x28, 
-0xfb, 0xa8, 0x22, 0x28, 0xfb, 0xac, 0x24, 0x3c, 
-0x76, 0x02, 0x10, 0x03, 0x26, 0x3c, 0x26, 0x1f, 
-0x4e, 0x75, 0x92, 0x83, 0x91, 0x82, 0x66, 0x06, 
-0x41, 0xe8, 0xfb, 0xc6, 0x60, 0x08, 0x30, 0x7c, 
-0xff, 0xaa, 0xd1, 0xed, 0x0a, 0x4c, 0x20, 0x50, 
-0x41, 0xe8, 0xfe, 0xf6, 0x4e, 0x90, 0x4c, 0xee, 
-0x04, 0xf8, 0xef, 0x9c, 0x4e, 0x5e, 0x4e, 0x75, 
-0x48, 0xe7, 0x1c, 0x38, 0x26, 0x48, 0x38, 0x01, 
-0x2a, 0x09, 0x28, 0x48, 0x18, 0xc0, 0xe0, 0x48, 
-0x18, 0x80, 0x17, 0x41, 0x00, 0x02, 0x30, 0x01, 
-0xe0, 0x48, 0x11, 0x40, 0x00, 0x03, 0x20, 0x6d, 
-0x0f, 0x10, 0x20, 0x28, 0xfb, 0xa8, 0x22, 0x28, 
-0xfb, 0xac, 0x24, 0x3c, 0x76, 0x02, 0x10, 0x03, 
-0x26, 0x3c, 0x26, 0x1f, 0x4e, 0x75, 0x92, 0x83, 
-0x91, 0x82, 0x66, 0x06, 0x41, 0xe8, 0xfb, 0xc6, 
-0x60, 0x08, 0x30, 0x7c, 0xff, 0xaa, 0xd1, 0xed, 
-0x0a, 0x4c, 0x20, 0x50, 0x41, 0xe8, 0xfe, 0x12, 
-0x48, 0x78, 0x07, 0xd0, 0x48, 0x78, 0x00, 0x04, 
-0x2f, 0x0b, 0x4e, 0x90, 0x4f, 0xef, 0x00, 0x0c, 
-0x4a, 0x40, 0x66, 0x00, 0x00, 0xa4, 0x20, 0x6d, 
-0x0f, 0x10, 0x20, 0x28, 0xfb, 0xa8, 0x22, 0x28, 
-0xfb, 0xac, 0x24, 0x3c, 0x76, 0x02, 0x10, 0x03, 
-0x26, 0x3c, 0x26, 0x1f, 0x4e, 0x75, 0x92, 0x83, 
-0x91, 0x82, 0x66, 0x06, 0x41, 0xe8, 0xfb, 0xc6, 
-0x60, 0x08, 0x30, 0x7c, 0xff, 0xaa, 0xd1, 0xed, 
-0x0a, 0x4c, 0x20, 0x50, 0x41, 0xe8, 0xfe, 0x12, 
-0x48, 0x78, 0x07, 0xd0, 0x3f, 0x04, 0x42, 0x67, 
-0x2f, 0x05, 0x4e, 0x90, 0x4f, 0xef, 0x00, 0x0c, 
-0x4a, 0x40, 0x66, 0x5c, 0x45, 0xfa, 0x00, 0x60, 
-0x70, 0x04, 0x20, 0x4b, 0x4e, 0x92, 0x36, 0x00, 
-0x30, 0x04, 0x20, 0x45, 0x4e, 0x92, 0xd6, 0x40, 
-0x16, 0x83, 0xe0, 0x4b, 0x18, 0x83, 0x20, 0x6d, 
-0x0f, 0x10, 0x20, 0x28, 0xfb, 0xa8, 0x22, 0x28, 
-0xfb, 0xac, 0x24, 0x3c, 0x76, 0x02, 0x10, 0x03, 
-0x26, 0x3c, 0x26, 0x1f, 0x4e, 0x75, 0x92, 0x83, 
-0x91, 0x82, 0x66, 0x06, 0x41, 0xe8, 0xfb, 0xc6, 
-0x60, 0x08, 0x30, 0x7c, 0xff, 0xaa, 0xd1, 0xed, 
-0x0a, 0x4c, 0x20, 0x50, 0x41, 0xe8, 0xfe, 0x12, 
-0x48, 0x78, 0x07, 0xd0, 0x48, 0x78, 0x00, 0x02, 
-0x2f, 0x0b, 0x4e, 0x90, 0x4f, 0xef, 0x00, 0x0c, 
-0x4c, 0xdf, 0x1c, 0x38, 0x4e, 0x75, 0x34, 0x00, 
-0x42, 0x41, 0x93, 0xc9, 0x60, 0x10, 0x70, 0x00, 
-0x30, 0x01, 0x10, 0x30, 0x08, 0x00, 0x02, 0x40, 
-0x00, 0xff, 0xd2, 0xc0, 0x52, 0x41, 0xb4, 0x41, 
-0x66, 0xec, 0x30, 0x09, 0x4e, 0x75, 0x31, 0x2e, 
-0x30, 0x32, 0x20, 0x55, 0x53, 0x42, 0x00, 0x52, 
-0x6f, 0x6d, 0x44, 0x75, 0x6d, 0x70, 0x65, 0x72, 
-0x20, 0x76, 0x25, 0x73, 0x00, 0x54, 0x79, 0x70, 
-0x65, 0x3a, 0x20, 0x48, 0x57, 0x25, 0x69, 0x00, 
-0x52, 0x4f, 0x4d, 0x20, 0x62, 0x61, 0x73, 0x65, 
-0x3a, 0x20, 0x30, 0x78, 0x25, 0x6c, 0x78, 0x00, 
-0x62, 0x79, 0x20, 0x54, 0x68, 0x65, 0x20, 0x54, 
-0x69, 0x4c, 0x50, 0x20, 0x54, 0x65, 0x61, 0x6d, 
-0x00, 0x53, 0x69, 0x7a, 0x65, 0x3a, 0x20, 0x25, 
-0x6c, 0x75, 0x20, 0x4b, 0x42, 0x00, 0x44, 0x6f, 
-0x6e, 0x65, 0x3a, 0x20, 0x25, 0x6c, 0x75, 0x2f, 
-0x25, 0x6c, 0x75, 0x4b, 0x42, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x00, 0x00, 0x20, 0x38, 0x00, 0xc8, 
-0x02, 0x80, 0x00, 0xe0, 0x00, 0x00, 0x20, 0x40, 
-0x22, 0x68, 0x01, 0x04, 0xd1, 0xfc, 0x00, 0x01, 
-0x00, 0x00, 0xb3, 0xc8, 0x64, 0x14, 0x0c, 0x51, 
-0x00, 0x16, 0x63, 0x0e, 0x4e, 0x4c, 0x46, 0xfc, 
-0x30, 0x00, 0x40, 0xc1, 0x46, 0xc0, 0xe7, 0x49, 
-0x6a, 0x04, 0x70, 0x01, 0x4e, 0x75, 0x20, 0x29, 
-0x00, 0x16, 0x4e, 0x75, 0x00, 0x00, 0x05, 0xf8, 
-0x01, 0x80, 0xf3, 0x3a, 0x19, 
 };
-static unsigned int romDumpSize89t = 1733;
+static unsigned int romDumpSize89t = 0;
--- a/src/rom92f2.h
+++ b/src/rom92f2.h
@@ -1,162 +1,3 @@
 static unsigned char romDump92[] = {
-0x2a, 0x2a, 0x54, 0x49, 0x39, 0x32, 0x2a, 0x2a, 
-0x01, 0x00, 0x6d, 0x61, 0x69, 0x6e, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x01, 0x00, 0x52, 0x00, 0x00, 0x00, 
-0x72, 0x6f, 0x6d, 0x64, 0x75, 0x6d, 0x70, 0x00, 
-0x12, 0x00, 0x00, 0x00, 0xf2, 0x04, 0x00, 0x00, 
-0xa5, 0x5a, 0x00, 0x00, 0x00, 0x00, 0x04, 0x98, 
-0x00, 0x32, 0x45, 0x58, 0x45, 0x20, 0x41, 0x50, 
-0x50, 0x4c, 0x00, 0x00, 0x04, 0x4e, 0x04, 0x52, 
-0x04, 0x54, 0x00, 0x1a, 0x00, 0x00, 0x00, 0x01, 
-0x00, 0x01, 0x00, 0x1e, 0x4f, 0xef, 0xef, 0xb4, 
-0x48, 0xe7, 0x1e, 0x38, 0x23, 0xfc, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x04, 0x46, 0x20, 0x3c, 
-0x00, 0x00, 0x00, 0x00, 0x60, 0x02, 0x54, 0x80, 
-0x20, 0x40, 0x0c, 0x90, 0x00, 0x00, 0x00, 0x00, 
-0x66, 0xf4, 0x23, 0xc0, 0x00, 0x00, 0x04, 0x4a, 
-0x72, 0xe4, 0xd2, 0x80, 0x23, 0xc1, 0x00, 0x00, 
-0x04, 0x46, 0x60, 0x02, 0x54, 0x80, 0x20, 0x40, 
-0x0c, 0x90, 0x00, 0x00, 0x00, 0x00, 0x66, 0xf4, 
-0x06, 0x80, 0xff, 0xff, 0xff, 0x6a, 0x23, 0xc0, 
-0x00, 0x00, 0x04, 0x4a, 0x48, 0x78, 0x0f, 0x00, 
-0x42, 0x67, 0x48, 0x79, 0x00, 0x00, 0x00, 0x00, 
-0x4e, 0xb9, 0x00, 0x00, 0x00, 0x00, 0x3f, 0x3c, 
-0x00, 0x02, 0x4e, 0xb9, 0x00, 0x00, 0x00, 0x00, 
-0x48, 0x7a, 0x03, 0x34, 0x48, 0x7a, 0x03, 0x35, 
-0x45, 0xef, 0x10, 0x56, 0x2f, 0x0a, 0x49, 0xf9, 
-0x00, 0x00, 0x00, 0x00, 0x4e, 0x94, 0x3f, 0x3c, 
-0x00, 0x01, 0x2f, 0x0a, 0x42, 0xa7, 0x47, 0xf9, 
-0x00, 0x00, 0x00, 0x00, 0x4e, 0x93, 0x4f, 0xef, 
-0x00, 0x22, 0x20, 0x39, 0x00, 0x00, 0x00, 0xc4, 
-0x06, 0x80, 0x00, 0x0f, 0xff, 0xff, 0x42, 0x40, 
-0x48, 0x40, 0xe8, 0x48, 0x3f, 0x00, 0x48, 0x7a, 
-0x03, 0x09, 0x2f, 0x0a, 0x4e, 0x94, 0x3f, 0x3c, 
-0x00, 0x01, 0x2f, 0x0a, 0x48, 0x78, 0x00, 0x14, 
-0x4e, 0x93, 0x48, 0x79, 0x00, 0x00, 0x00, 0x00, 
-0x48, 0x7a, 0x02, 0xfe, 0x2f, 0x0a, 0x4e, 0x94, 
-0x4f, 0xef, 0x00, 0x1e, 0x3e, 0xbc, 0x00, 0x01, 
-0x2f, 0x0a, 0x48, 0x78, 0x00, 0x28, 0x4e, 0x93, 
-0x48, 0x7a, 0x02, 0xf6, 0x2f, 0x0a, 0x4e, 0x94, 
-0x3f, 0x3c, 0x00, 0x01, 0x2f, 0x0a, 0x48, 0x78, 
-0x00, 0x50, 0x4e, 0x93, 0x4f, 0xef, 0x00, 0x1c, 
-0x48, 0x78, 0x00, 0x14, 0x48, 0x78, 0x00, 0x04, 
-0x45, 0xef, 0x00, 0x26, 0x2f, 0x0a, 0x20, 0x7a, 
-0x03, 0x28, 0x4e, 0x90, 0x4f, 0xef, 0x00, 0x0c, 
-0x4a, 0x40, 0x66, 0x00, 0x02, 0x08, 0x1a, 0x2f, 
-0x00, 0x1f, 0x1c, 0x2f, 0x00, 0x1e, 0x16, 0x2f, 
-0x00, 0x21, 0xe1, 0x4b, 0x42, 0x40, 0x10, 0x2f, 
-0x00, 0x20, 0x86, 0x40, 0x47, 0xfa, 0x02, 0xde, 
-0x70, 0x04, 0x20, 0x4a, 0x4e, 0x93, 0x78, 0x00, 
-0x38, 0x03, 0x48, 0x78, 0x00, 0x14, 0x2f, 0x04, 
-0x2f, 0x0a, 0x20, 0x7a, 0x02, 0xec, 0x4e, 0x90, 
-0x4f, 0xef, 0x00, 0x0c, 0x4a, 0x40, 0x66, 0x20, 
-0x48, 0x78, 0x00, 0x14, 0x48, 0x78, 0x00, 0x02, 
-0x48, 0x72, 0x48, 0x00, 0x20, 0x7a, 0x02, 0xd2, 
-0x4e, 0x90, 0x4f, 0xef, 0x00, 0x0c, 0x4a, 0x40, 
-0x66, 0x06, 0x30, 0x03, 0x20, 0x4a, 0x4e, 0x93, 
-0x30, 0x39, 0x00, 0x00, 0x00, 0x1c, 0x56, 0xc0, 
-0x18, 0x00, 0x48, 0x84, 0x44, 0x44, 0x32, 0x05, 
-0xe1, 0x49, 0x42, 0x40, 0x10, 0x06, 0x82, 0x40, 
-0x0c, 0x41, 0x00, 0x02, 0x67, 0x20, 0x0c, 0x41, 
-0x00, 0x03, 0x67, 0x2c, 0x0c, 0x41, 0x00, 0x05, 
-0x67, 0x00, 0x00, 0x96, 0x0c, 0x41, 0xaa, 0x55, 
-0x66, 0x00, 0x01, 0x72, 0x93, 0xc9, 0x42, 0x41, 
-0x70, 0x01, 0x60, 0x00, 0x01, 0x62, 0x93, 0xc9, 
-0x42, 0x41, 0x70, 0x02, 0x41, 0xef, 0x00, 0x1e, 
-0x61, 0x00, 0x01, 0x7c, 0x60, 0x00, 0x01, 0x6e, 
-0x20, 0x39, 0x00, 0x00, 0x00, 0xc4, 0x06, 0x80, 
-0x00, 0x0f, 0xff, 0xff, 0x22, 0x00, 0x02, 0x81, 
-0x00, 0xf0, 0x00, 0x00, 0xe0, 0x89, 0x43, 0xef, 
-0x10, 0x68, 0x42, 0x40, 0x48, 0x40, 0xe0, 0x48, 
-0x80, 0x81, 0x23, 0x00, 0x72, 0x04, 0x70, 0x03, 
-0x41, 0xef, 0x00, 0x1e, 0x61, 0x00, 0x01, 0x48, 
-0x20, 0x39, 0x00, 0x00, 0x00, 0xc4, 0x06, 0x80, 
-0x00, 0x0f, 0xff, 0xff, 0x72, 0x0a, 0xe2, 0xa8, 
-0x02, 0x80, 0x00, 0x3f, 0xfc, 0x00, 0x2f, 0x00, 
-0x48, 0x7a, 0x01, 0xdf, 0x45, 0xef, 0x10, 0x2c, 
-0x2f, 0x0a, 0x4e, 0xb9, 0x00, 0x00, 0x00, 0x00, 
-0x3f, 0x3c, 0x00, 0x04, 0x2f, 0x0a, 0x48, 0x78, 
-0x00, 0x3c, 0x4e, 0xb9, 0x00, 0x00, 0x00, 0x00, 
-0x4f, 0xef, 0x00, 0x16, 0x60, 0x00, 0x00, 0xe6, 
-0x76, 0x00, 0x16, 0x2f, 0x00, 0x1f, 0xe1, 0x8b, 
-0x86, 0x2f, 0x00, 0x1e, 0x10, 0x2f, 0x00, 0x21, 
-0xe1, 0x48, 0x48, 0x40, 0x42, 0x40, 0x72, 0x00, 
-0x12, 0x2f, 0x00, 0x20, 0x48, 0x41, 0x42, 0x41, 
-0x80, 0x81, 0x86, 0x80, 0x20, 0x39, 0x00, 0x00, 
-0x00, 0xc4, 0x06, 0x80, 0x00, 0x0f, 0xff, 0xff, 
-0x02, 0x80, 0xff, 0xf0, 0x00, 0x00, 0xb0, 0x83, 
-0x64, 0x0e, 0x93, 0xc9, 0x42, 0x41, 0x70, 0x04, 
-0x41, 0xef, 0x00, 0x1e, 0x61, 0x00, 0x00, 0xc0, 
-0x20, 0x39, 0x00, 0x00, 0x00, 0xc4, 0x06, 0x80, 
-0x00, 0x0f, 0xff, 0xff, 0x72, 0x0a, 0xe2, 0xa8, 
-0x02, 0x80, 0x00, 0x3f, 0xfc, 0x00, 0x2f, 0x00, 
-0x20, 0x03, 0xe2, 0xa8, 0x2f, 0x00, 0x48, 0x7a, 
-0x01, 0x5e, 0x45, 0xef, 0x10, 0x30, 0x2f, 0x0a, 
-0x4e, 0xb9, 0x00, 0x00, 0x00, 0x00, 0x3f, 0x3c, 
-0x00, 0x04, 0x2f, 0x0a, 0x48, 0x78, 0x00, 0x3c, 
-0x4e, 0xb9, 0x00, 0x00, 0x00, 0x00, 0x22, 0x43, 
-0xd3, 0xfc, 0x00, 0x00, 0x00, 0x00, 0x10, 0x11, 
-0x72, 0x00, 0x4f, 0xef, 0x00, 0x1a, 0x20, 0x43, 
-0xd1, 0xfc, 0x00, 0x00, 0x00, 0x00, 0xb0, 0x30, 
-0x18, 0x00, 0x66, 0x34, 0x34, 0x01, 0x52, 0x42, 
-0x52, 0x81, 0x0c, 0x81, 0x00, 0x00, 0x10, 0x00, 
-0x66, 0xe4, 0x0c, 0x42, 0x10, 0x00, 0x66, 0x20, 
-0x02, 0x40, 0x00, 0xff, 0x42, 0x2f, 0x10, 0x60, 
-0x1f, 0x7c, 0x00, 0x10, 0x10, 0x61, 0x1f, 0x40, 
-0x10, 0x62, 0x42, 0x2f, 0x10, 0x63, 0x43, 0xef, 
-0x10, 0x60, 0x72, 0x04, 0x70, 0x07, 0x60, 0x06, 
-0x32, 0x3c, 0x10, 0x00, 0x70, 0x06, 0x41, 0xef, 
-0x00, 0x1e, 0x61, 0x22, 0x4a, 0x44, 0x66, 0x14, 
-0x60, 0x00, 0xfd, 0xde, 0x30, 0x39, 0x00, 0x00, 
-0x00, 0x1c, 0x56, 0xc0, 0x18, 0x00, 0x48, 0x84, 
-0x44, 0x44, 0x60, 0xe8, 0x4c, 0xdf, 0x1c, 0x78, 
-0x4f, 0xef, 0x10, 0x4c, 0x4e, 0x75, 0x48, 0xe7, 
-0x1c, 0x38, 0x26, 0x48, 0x38, 0x01, 0x2a, 0x09, 
-0x28, 0x48, 0x18, 0xc0, 0xe0, 0x48, 0x18, 0x80, 
-0x17, 0x41, 0x00, 0x02, 0x30, 0x01, 0xe0, 0x48, 
-0x11, 0x40, 0x00, 0x03, 0x48, 0x78, 0x00, 0x04, 
-0x2f, 0x08, 0x20, 0x7a, 0x00, 0xd0, 0x4e, 0x90, 
-0x50, 0x8f, 0x4a, 0x40, 0x66, 0x3a, 0x3f, 0x04, 
-0x42, 0x67, 0x2f, 0x05, 0x20, 0x7a, 0x00, 0xbe, 
-0x4e, 0x90, 0x50, 0x8f, 0x4a, 0x40, 0x66, 0x28, 
-0x45, 0xfa, 0x00, 0x92, 0x70, 0x04, 0x20, 0x4b, 
-0x4e, 0x92, 0x36, 0x00, 0x30, 0x04, 0x20, 0x45, 
-0x4e, 0x92, 0xd6, 0x40, 0x16, 0x83, 0xe0, 0x4b, 
-0x18, 0x83, 0x48, 0x78, 0x00, 0x02, 0x2f, 0x0b, 
-0x20, 0x7a, 0x00, 0x92, 0x4e, 0x90, 0x50, 0x8f, 
-0x4c, 0xdf, 0x1c, 0x38, 0x4e, 0x75, 0x31, 0x2e, 
-0x30, 0x32, 0x00, 0x52, 0x6f, 0x6d, 0x44, 0x75, 
-0x6d, 0x70, 0x65, 0x72, 0x20, 0x76, 0x25, 0x73, 
-0x00, 0x54, 0x79, 0x70, 0x65, 0x3a, 0x20, 0x54, 
-0x49, 0x2d, 0x39, 0x32, 0x20, 0x25, 0x69, 0x00, 
-0x52, 0x4f, 0x4d, 0x20, 0x62, 0x61, 0x73, 0x65, 
-0x3a, 0x20, 0x30, 0x78, 0x25, 0x6c, 0x78, 0x00, 
-0x62, 0x79, 0x20, 0x54, 0x68, 0x65, 0x20, 0x54, 
-0x69, 0x4c, 0x50, 0x20, 0x54, 0x65, 0x61, 0x6d, 
-0x00, 0x53, 0x69, 0x7a, 0x65, 0x3a, 0x20, 0x25, 
-0x6c, 0x75, 0x20, 0x4b, 0x42, 0x00, 0x44, 0x6f, 
-0x6e, 0x65, 0x3a, 0x20, 0x25, 0x6c, 0x75, 0x2f, 
-0x25, 0x6c, 0x75, 0x4b, 0x42, 0x20, 0x20, 0x20, 
-0x20, 0x20, 0x00, 0x00, 0x34, 0x00, 0x42, 0x41, 
-0x93, 0xc9, 0x60, 0x10, 0x70, 0x00, 0x30, 0x01, 
-0x10, 0x30, 0x08, 0x00, 0x02, 0x40, 0x00, 0xff, 
-0xd2, 0xc0, 0x52, 0x41, 0xb4, 0x41, 0x66, 0xec, 
-0x30, 0x09, 0x4e, 0x75, 0x00, 0x00, 0x00, 0x00, 
-0x00, 0x00, 0x00, 0x00, 0x8a, 0xb3, 0x0c, 0x00, 
-0x00, 0x00, 0x00, 0x01, 0x04, 0x89, 0x09, 0x14, 
-0x00, 0x02, 0x23, 0x00, 0x05, 0x42, 0xc0, 0x49, 
-0x46, 0x00, 0x01, 0x4a, 0xc0, 0x49, 0x46, 0x00, 
-0x02, 0x38, 0x00, 0x09, 0xc0, 0x3b, 0xc0, 0x55, 
-0x00, 0x01, 0x30, 0x00, 0x09, 0x50, 0x13, 0x7e, 
-0x17, 0x31, 0x11, 0x1f, 0x07, 0x00, 0x0a, 0x09, 
-0x04, 0x00, 0x03, 0x33, 0x00, 0x00, 0x00, 0x74, 
-0x69, 0x6f, 0x73, 0x00, 0xe9, 0x12, 0xe4, 0x00, 
-0xe8, 0x19, 0xe4, 0xe5, 0x00, 0x00, 0x40, 0xdc, 
-0x3b, 0x37, 
 };
-static unsigned int romDumpSize92 = 1266;
+static unsigned int romDumpSize92 = 0;
